<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <title>Test Components</title>
    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Montserrat:400,700" />
    <link rel="stylesheet" href="http://modelseed.org/css/core.css" />
    <link rel="stylesheet" href="css/bootstrap.css" />
    <link rel="stylesheet" href="css/dataTables.bootstrap4.min.css" />
    <link rel="stylesheet" href="css/select2.min.css" />
    <link rel="stylesheet" href="css/fontawesome.min.css" />
    <link rel="stylesheet" href="css/solid.min.css" />
    <link rel="stylesheet" href="css/regular.min.css" />
    <link rel="stylesheet" href="css/seed.css" />
    <style>
        .switch-toggle {
  width: 10em;
}

        .report-table tbody:hover td {
          background:#CCC;
          cursor:pointer;
        }

.switch-toggle label:not(.disabled) {
  cursor: pointer;
}
        
@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro);
*{
  margin:0px;
  padding:0px;
}
html{
  
}

        .border_test {
            border: 1px solid black;
        }
.wrapper {
  margin: 100px 0;
  background: #497dd0;
}
.toggle_radio{
  position: relative;
  background: rgba(255,255,255,.1);
  margin: 4px auto;
  overflow: hidden;
  padding: 0 !important;
  -webkit-border-radius: 50px;
  -moz-border-radius: 50px;
  border-radius: 50px;
  position: relative;
  height: 26px;
  width: 518px;
}
.toggle_radio > * {
  float: left;
}
.toggle_radio input[type=radio]{
  display: none;
  /*position: fixed;*/
}
.toggle_radio label{
  font: 90%/1.618 "Source Sans Pro";
  color: rgba(255,255,255,.9);
  z-index: 0;
  display: block;
  width: 100px;
  height: 20px;
  margin: 3px 3px;
  -webkit-border-radius: 50px;
  -moz-border-radius: 50px;
  border-radius: 50px;
  cursor: pointer;
  z-index: 1;
  /*background: rgba(0,0,0,.1);*/
  text-align: center;
  /*margin: 0 2px;*/
  /*background: blue;*/ /*make it blue*/
}
.toggle_option_slider{
  /*display: none;*/
  /*background: red;*/
  width: 100px;
  height: 20px;
  position: absolute;
  top: 3px;
  -webkit-border-radius: 50px;
  -moz-border-radius: 50px;
  border-radius: 50px;
  -webkit-transition: all .4s ease;
  -moz-transition: all .4s ease;
  -o-transition: all .4s ease;
  -ms-transition: all .4s ease;
  transition: all .4s ease;
}


#first_toggle:checked ~ .toggle_option_slider{
  background: rgba(255,255,255,.3);
  left: 3px;
}
#second_toggle:checked ~ .toggle_option_slider{
  background: rgba(255,255,255,.3);
  left: 109px;
}
#third_toggle:checked ~ .toggle_option_slider{
  background: rgba(255,255,255,.3);
  left: 215px;
}
#a_toggle:checked ~ .toggle_option_slider{
  background: rgba(255,255,255,.3);
  left: 325px;
}
        .select2-container .select2-selection--single .select2-selection__rendered {
          z-index:1;
        }
        
        .glyphicon {
            -moz-osx-font-smoothing:grayscale;
            -webkit-font-smoothing:antialiased;
            display:inline-block;
            font-style:normal;
            font-variant:normal;
            text-rendering:auto;
            line-height:1;
            font-family:"Font Awesome 5 Free";
            font-weight:900;
        }
        .glyphicon-ok:before {
            content:"\f00c"
        }
        
        .glyphicon-plus-sign:before {
            content:"\f067"
        }
        
        .glyphicon-minus-sign:before {
            content:"\f068"
        }
        
        .glyphicon-resize-full:before {
            content:"\f337"
        }
    </style>
  </head>
<body>
<div class="md-toolbar-tools md-whiteframe-z2 app-toolbar ng-scope md-default-theme">
  <div class="md-toolbar-tools ng-scope">
    <a class="md-button md-default-theme" ng-transclude="" flex="22" ui-sref="app.RefModels" href="#">
      <span style="font-size: 18pt;" class="ng-scope">Reference Data</span>
    </a>
  </div>
</div>



<div>
  <!--
  <div class="container">
  <iframe src="http://modelseed.org/biochem/reactions/rxn00148" style="width: 100%; height: 500px">

  </iframe>
  -->
  <div class="input-group mb-3">
    <div class="input-group-prepend">
      <span class="input-group-text">Add</span>
    </div>
    <input type="text" class="form-control" aria-label="Amount (to the nearest dollar)">
    <div class="input-group-append">
      <button class="btn btn-outline-secondary" type="button">Button</button>
    </div>
  </div>

  </div>

<div>
  <i class="fas fa-bug"></i>
  <i class="fas fa-book"></i>
  <i class="fas fa-dna"></i> Sequence
  <i class="fas fa-cog"></i> Automated
  <i class="fas fa-flask"></i> Experimental
  <i class="fas fa-vial"></i> Essay
  <i class="fas fa-laptop"></i> Computational
  <i class="fas fa-user-check"></i> Human
  <i class="fas fa-pastafarianism"></i> Mutant
  <i class="fas fa-ban"></i> Blocked

</div>


  <div id="omgtest">
    <div class="seed-annotation-group">

      <div class="float-left">
        <div class="btn-group btn-group-toggle seed-annotation-curated-group" data-toggle="buttons">
          <div class="seed-annotation-control-label">fliu</div>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star-half-alt"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="far fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-ban"></i></label>
          <label class="btn btn-seed-sm btn-warning active"><input type="radio" name="options" autocomplete="off"><i class="fas fa-question"></i></label>
        </div>
      </div>
      <div class="float-left">
        <div>
          Phosphohistidine swiveling domain <i class="fas fa-spinner fa-spin"></i>
        </div>
      </div>
      <div class="float-left">
        <div>
          <span class="label label-annotation-source">BETA_RAST (1306/1558)</span>
        </div>
      </div>


    </div>

    <div class="seed-annotation-group">

      <div class="float-left">
        <div class="btn-group btn-group-toggle seed-annotation-auto-group" data-toggle="buttons">
          <div class="seed-annotation-control-label">system</div>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star-half-alt"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="far fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-ban"></i></label>
          <label class="btn btn-seed-sm btn-warning active"><input type="radio" name="options" autocomplete="off"><i class="fas fa-question"></i></label>
        </div>
      </div>
      opt2  <span class="badge badge-danger"><i class="fas fa-bug"></i> Error: rxn000001</span>
      Download SVGPerfect for when you want to use just one icon as a vector on the desktop or in your own icon workflow.
    </div>

    <div class="seed-annotation-group">

      <div class="float-left">
        <div class="btn-group btn-group-toggle seed-annotation-auto-group" data-toggle="buttons">
          <div class="seed-annotation-control-label">algorithm</div>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-star-half-alt"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="far fa-star"></i></label>
          <label class="btn btn-seed-sm btn-warning"><input type="radio" name="options" autocomplete="off"><i class="fas fa-ban"></i></label>
          <label class="btn btn-seed-sm btn-warning active"><input type="radio" name="options" autocomplete="off"><i class="fas fa-question"></i></label>
        </div>
      </div>
      opt3
    </div>
  </div>
  <div style="clear:both;"></div>
</div>

    check-circle
    <span class="dropdown-button-icon glyphicon glyphicon-ok">!!</span>
    <i class="fas fa-check">glyphicon-ok</i>
    <i class="fas fa-times">glyphicon-remove</i>
    <i class="fas fa-times-circle">glyphicon-remove</i>
    
    <i class="fas fa-plus">glyphicon-plus-sign</i>
    <i class="fas fa-minus">glyphicon-minus-sign</i>
    <i class="fas fa-arrows-alt-h">glyphicon-resize-full</i>
    
    <i class="fas fa-arrows-alt">glyphicon-move</i>
    <i class="fas fa-mouse-pointer">glyphicon-hand-up</i>
    <i class="fas fa-wrench">glyphicon-plus</i>
    <i class="fas fa-undo">glyphicon-repeat</i>
    <i class="fas fa-font">glyphicon-font</i>
    
    <i class="fas fa-arrow-left">glyphicon-arrow-left</i>
    <i class="fas fa-arrow-right">glyphicon-arrow-right</i>
    <i class="fas fa-arrow-up">glyphicon-arrow-up</i>
    <i class="fas fa-arrow-down">glyphicon-arrow-down</i>   


<div class="container-fluid">
  <div id="test_container1" class="border_test">
    Test Container 1
    <table class="table">
      <thead>
      <tr>
        <th>Gene Hit</th>
        <th></th>
        <th>Identity/ <br> Coverage</th>
        <th></th>
        <th>Organism</th>
        <th>Title</th>
        <th>Text</th>
        <th>Authors</th>
        <th></th>
      </tr>
      </thead>
      <tbody>
      <tr>
        <td>SOL4_YEAST / P53315 <br> Uniprot</td>
        <td></td>
        <td>100% / 100%</td>
        <td></td>
        <td>Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast)</td>
        <td>Global translational impacts of the loss of the tRNA modification t6A in yeast <br>
          <small>Microbial cell (Graz, Austria) 2016</small> <br>
          <small>Thiaville PC, Legendre R, Rojas-Benítez D, Baudin-Baillieu A, Hatin I, Chalancon G, Glavic A, Namy O, de Crécy-Lagard V.</small>
        </td>
        <td>“...genes involved in growth on non-fermentable carbon sources (see summary in SGD 98 ), Sol4 (YGR248W), which functions in the pentose phosphate pathway (see summary in SGD 98 ), and Smc6 (YLR383W), a component of the SMC5-SMC6 complex that plays a key role in the removal...”</td>
        <td></td>
        <td><div style="width: 450px">

          <div class="js-example-basic-multiple" ></div>
        </div></td>


      </tr>
      </tbody>

    </table>
  </div>
</div>

    <div class="container">
<div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-secondary active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked> Active
  </label>
  <label class="btn btn-seed-sm btn-danger">
    <input type="radio" name="options" id="option2" autocomplete="off"> Radio
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"> Radio
  </label>
  <label class="btn btn-seed-sm btn-success">
    <input type="radio" name="options" id="option3" autocomplete="off"> Radio
  </label>
</div>
GCF_000146045.2 (YAL038W or YOR347C) {'YAL038W', 'YOR347C'}
YAL038W pyruvate kinase CDC19 cluster308
YOR347C pyruvate kinase PYK2 cluster308
GCF_000182925.2 NCU06075 {'NCU06075'}
NCU06075 pyruvate kinase cluster308
GCF_000226395.1 ( Pc18g06000 ) {'Pc18g06000'}
Pc18g06000 hypothetical protein cluster308



      <p></p>

<div id="test_tooltip1" class="border_test">
    Test Tooltip 1
</div>
        
    <p></p>

<div id="test_tooltip2" class="border_test">
    Test Tooltip 2
</div>
        
    <p></p>
        
<div id="test_tooltip3" class="border_test">
    Test Tooltip 3
</div>
        
    <p></p>
        
<div id="test_tooltip4" class="border_test">
    Test Tooltip 4
</div>
        
    <p></p>
        
<div id="test_tooltip5" class="border_test">
    Test Tooltip 5
</div>
        
    <p></p>
        
<div class="border_test">
    <div><span class="badge badge-success">GCF_000146045.2</span> Saccharomyces cerevisiae S288c</div>
    <div>iMM904 : (<span style="color:red">YAL038W</span> and <span style="color:teal">YOR347C</span>)</div>
    <div>iTO11111 : (<span style="color:teal">YOR347C</span>)</div>
    <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> <span class="badge badge-danger">YAL038W</span>
     pyruvate kinase CDC19
        </div>
    
    <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> <span class="badge badge-info">YOR347C</span>
     pyruvate kinase PYK2
        </div>
    
    <div><span class="badge badge-success">GCF_000182925.2</span> Neurospora crassa OR74A</div>
    <div>iJDZ836 : <span style="color:teal">NCU06075</span></div>
    
    <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label></div> <span class="badge badge-info">NCU06075</span> pyruvate kinase
        </div>
    
    <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label></div> <span class="badge badge-danger">NCU02111</span> pyruvate kinase other
        </div>
    

    <div><span class="badge badge-success">GCF_000226395.1</span> Penicillium rubens Wisconsin 54-1255</div>
    <div>iAL1006 : ( Pc18g06000 )</div>
    
    <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> <span class="badge badge-info">Pc18g06000</span>
     hypothetical protein
        </div>
    
    <span class="badge badge-info">cluster 308</span>
    <span class="badge badge-danger">cluster 121</span>
</div>
        
        <p></p>
        <!-- tool_tip example3 -->
        
<div class="border_test" style="width : 500px">
    <table class="table table-striped">
        <thead class="thead-light">
        <tr>
            <th scope="col"></th>
            <th scope="col"><span class="badge badge-info">cluster 308</span></th>
            <th scope="col"><span class="badge badge-danger">cluster 121</span></th>
        </tr>
            </thead>
        
<tr>
            <td><span class="badge badge-success">GCF_000146045.2</span><br>Saccharomyces cerevisiae S288c</td>
            <td><div><span class="badge badge-info">YOR347C</span> pyruvate kinase PYK2</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
            <td>
                <div><span class="badge badge-danger">YAL038W</span> pyruvate kinase CDC19</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
        </tr>
        
<tr>
            <td></td>
            <td><div><span class="badge badge-info">YAL038W</span> pyruvate kinase CDC19</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
            <td>
            </td>
        </tr>
        
        <tr>
            <td><span class="badge badge-success">GCF_000182925.2</span><br>Neurospora crassa OR74A</td>
            <td><div><span class="badge badge-info">NCU06075</span> pyruvate kinase</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
            <td>
                <div><span class="badge badge-danger">NCU02111</span> pyruvate kinase other</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
        </tr>
        <tr>
            <td><span class="badge badge-success">GCF_000226395.1</span><br>Penicillium rubens Wisconsin 54-1255</td>
            <td><div><span class="badge badge-info">Pc18g06000</span>
     hypothetical protein</div>
                <div>
    <div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked><i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off"><i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off"><i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off"><i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off"><i class="fas fa-question"></i>
  </label>
    </div> 
        </div>
            </td>
            <td></td>
        </tr>
    </table>
</div>

      <br/>

      <div class="border_test report-table">
        <table class="table">
          <thead class="thead-light">
            <tr>
              <th scope="col">ModelSEED</th>
              <th scope="col"></th>
              <th scope="col">iMM904</th>
              <th scope="col">iJDZ</th>
              <th scope="col">iAL</th>
            </tr>
          </thead>
          <tbody>
          <tr>
            <td rowspan="2">
              <a href="javascript:window.open('http://modelseed.org/biochem/compounds/cpd00027','blank')" target="">cpd00027</a>
              Edit
            </td>
            <td>c</td>
            <td>M_33434</td>
            <td>M_sdsd</td>
            <td>M_sdsd</td>
          </tr>
          <tr>
            <td>m</td>
            <td>M_33434</td>
            <td>M_sdsd</td>
            <td>M_sdsd</td>
          </tr>
          </tbody>
          <tbody>
            <tr>
              <td rowspan="2">
                <a href="http://modelseed.org/biochem/compounds/cpd00111" target="_blank">cpd00111</a>
                Edit
              </td>
              <td>c</td>
              <td>M_33434</td>
              <td>M_sdsd</td>
              <td>M_sdsd</td>
            </tr>
            <tr>
              <td>m</td>
              <td>M_33434</td>
              <td>M_sdsd</td>
              <td>M_sdsd</td>
            </tr>
          </tbody>
          <tbody>
            <tr>
              <td rowspan="2">
                <select>
                  <option value="cpd00222" selected>cpd00222</option>
                  <option value="cpd00223" selected>cpd00223</option>
                  <option value="cpd00224" selected>cpd00224</option>
                </select>
              </td>
              <td>c</td>
              <td>M_33434</td>
              <td>M_sdsd</td>
              <td>M_sdsd</td>
            </tr>
            <tr>
              <td>m</td>
              <td>M_33434</td>
              <td>M_sdsd</td>
              <td>M_sdsd</td>
            </tr>
          </tbody>

        </table>
      </div>

<i class="fas fa-star"></i>
<i class="fas fa-star-half-alt"></i>
<i class="far fa-star"></i>
<i class="fas fa-ban"></i>
<i class="fas fa-question"></i>

<div class="btn-group btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-seed-sm btn-warning active">
    <input type="radio" name="options" id="option1" autocomplete="off" checked>
    <i class="fas fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option2" autocomplete="off">
    <i class="fas fa-star-half-alt"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option3" autocomplete="off">
    <i class="far fa-star"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option4" autocomplete="off">
    <i class="fas fa-ban"></i>
  </label>
  <label class="btn btn-seed-sm btn-warning">
    <input type="radio" name="options" id="option5" autocomplete="off">
    <i class="fas fa-question"></i>
  </label>
</div>

<i class="fas fa-star-half"></i>
<i class="far fa-star-half"></i>

<div class="switch-toggle switch-3 switch-candy">
    <input id="on" name="state-d" type="radio" checked="">
    <label for="on" onclick="">ON</label>

    <input id="na" name="state-d" type="radio" disabled checked="checked">
        <label for="na" onclick="">N/A</label>

    <input id="off" name="state-d" type="radio">
    <label for="off" onclick="">OFF</label>

    <a></a>
</div>

<div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
  <div class="btn-group mr-2" role="group" aria-label="First group">
    <button type="button" class="btn btn-secondary">1</button>
    <button type="button" class="btn btn-secondary">2</button>
    <button type="button" class="btn btn-secondary">3</button>
    <button type="button" class="btn btn-secondary">4</button>
  </div>
  <div class="btn-group mr-2" role="group" aria-label="Second group">
    <button type="button" class="btn btn-secondary">5</button>
    <button type="button" class="btn btn-secondary">6</button>
    <button type="button" class="btn btn-secondary">7</button>
  </div>
  <div class="btn-group" role="group" aria-label="Third group">
    <button type="button" class="btn btn-secondary">8</button>
  </div>
</div>



<div class="wrapper">
  <div class="toggle_radio">
    <input type="radio" class="toggle_option" id="first_toggle" name="toggle_option">
    <input type="radio" checked class="toggle_option" id="second_toggle" name="toggle_option">
    <input type="radio" class="toggle_option" id="third_toggle" name="toggle_option">
    <input type="radio" class="toggle_option" id="a_toggle" name="toggle_option">
    <label for="first_toggle"><p>First Button</p></label>
    <label for="second_toggle"><p>Second Button</p></label>
    <label for="third_toggle"><p>Third Button</p></label>
    <label for="a_toggle"><p>Another Button</p></label>
    <div class="toggle_option_slider"></div>
  </div>
</div>

<button>Toggle</button>
<p>Hello</p>
<p style="display: none">Good Bye 1</p>
<p style="display: none">Good Bye 2</p>
</div>

<!-- Modal -->
<div class="modal fade" id="modal_comment" tabindex="-1" role="dialog" aria-labelledby="modal_comment_label" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content" style="width: 500px">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalLabel">Comment</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <div class="form-group">
          <label id="modal_label_reaction">rxn00148: ATP + Pyruvate <=> ADP + Phosphoenolpyruvate + H+</label>
          <br>
          <label id="modal_label" for="text_comment">Star YAL038W: pyruvate kinase CDC19</label>
          <textarea class="form-control rounded-0" id="text_comment" rows="6"></textarea>
        </div>
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </div>
    </div>
  </div>
</div>
</body>
  <script src="js/jquery-3.4.1.min.js"></script>
  <script src="js/underscore-1.9.1.min.js"></script>
  <script src="js/select2.full.js"></script>
  <script src="js/chroma.min.js"></script>
  <script src="js/popper.min.js"></script>
  <script src="js/bootstrap.min.js"></script>
  <script src="js/jquery.dataTables.min.js"></script>
  <script src="js/dataTables.bootstrap4.min.js"></script>
  <script src="js/annotation.js"></script>
  <script src="js/tooltip_ortholog.js"></script>
  <script src="js/mock_data.js"></script>
  <script src="js/merge_report.js"></script>
  <script src="js/escher_tooltip_annotation.js"></script>
  <script src="js/widget_paper_blast.js"></script>
<script>
$( "button" ).click(function() {
  $( "p" ).toggle();
});
$(function() {

  let e = {'SOL4_YEAST / P53315':
      {'organism': "Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast)",
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4717488',
        'text': '“...genes involved in growth on non-fermentable carbon sources (see summary in SGD 98 ), Sol4 (YGR248W), which functions in the pentose phosphate pathway (see summary in SGD 98 ), and Smc6 (YLR383W), a component of the SMC5-SMC6 complex that plays a key role in the removal...”',
        'title': 'Global translational impacts of the loss of the tRNA modification t6A in yeast',
        'journal': 'Microbial cell (Graz, Austria) 2016',
        'authors': 'Thiaville PC, Legendre R, Rojas-Benítez D, Baudin-Baillieu A, Hatin I, Chalancon G, Glavic A, Namy O, de Crécy-Lagard V.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163424',
          'text': '“...0.024 0.222 R_AGAT_SC YDL052C 0.009 0.733 0.034 0.805 R_FTHFCLm YER183C 0.022 0 0.014 0 R_G6PDA YGR248W 0.017 0 0.007 0 R_G6PDA YHR163W 0.222 0 0.005 0 R_G6PDA YNR034W 0.023 0 0.028 0 R_GLUN YIL033C 0.079 0 0.205 0 R_M1PD YNR073C 0.016 0 0.024 0 R_MACACI YLL060C...”',
          'title': 'On the formalization and reuse of scientific research',
          'journal': 'Journal of the Royal Society, Interface 2011',
          'authors': 'King RD, Liakata M, Lu C, Oliver SG, Soldatova LN.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3063666',
          'text': '“...YGR138C YER130C YCR021C YDR171W YJL144W YGR249W YGR142W YGR248W YDR258C YOR134W YFL053W YPR158W YPR015C YGL037C YLL026W YNL077W YJL082W YBR214W YPL247C YNL007C...”',
          'title': 'New insight into the role of the Cdc34 ubiquitin-conjugating enzyme in cell cycle regulation via Ace2 and Sic1',
          'journal': 'Genetics 2011',
          'authors': 'Cocklin R, Heyen J, Larry T, Tyers M, Goebl M.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672114',
          'text': '“...TKL2 -3.23 -4.06 -4.16 9.50 19.33 PPP transketolase; needed for synthesis of aromatic amino acids YGR248W SOL4 -3.17 -2.15 -2.66 4.77 6.74 6-phosphogluconolactonase with similarity to Sol3p YPR151C SUE1 -3.17 -1.76 -1.45 5.52 5.45 Mitochondrial protein; degradation of unstable forms of cytochrome c YDR258C HSP78 -3.16...”',
          'title': 'Transcriptional response to mitochondrial NADH kinase deficiency in Saccharomyces cerevisiae',
          'journal': 'Mitochondrion 2009',
          'authors': 'Stuart GR, Humble MM, Strand MK, Copeland WC.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673449',
          'text': '“...xenobiotic agents and environmental or physiological stresses YPL054W LEE1 6.54 Zinc-finger protein of unknown function YGR248W SOL4 6.47 6-phosphogluconolactonase with similarity to Sol3p YEL033W YEL033W 6.42 Predicted to have metabolic role based on analysis of gene networks YLR162W YLR162W 6.38 Putative protein of unknown function; overexpression...”',
          'title': 'Construction and application of a protein and genetic interaction network (yeast interactome)',
          'journal': 'Nucleic acids research 2009',
          'authors': 'Stuart GR, Copeland WC, Strand MK.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2667776',
          'text': '“...YLL051C YMR315W YPL088W YHR209W YJL155C YDL022W YER062C YGR248W YGL157W YLR327C YEL060C YOR374W YMR105C YOL112W YMR238W YKL161C YHR030C YOR208W YIL117C YMR173W...”',
          'title': 'The high osmotic response and cell wall integrity pathways cooperate to regulate transcriptional responses to zymolyase-induced cell wall stress in Saccharomyces cerevisiae',
          'journal': 'The Journal of biological chemistry 2009',
          'authors': 'García R, Rodríguez-Peña JM, Bermejo C, Nombela C, Arroyo J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2254601',
          'text': '“...unknown unknown YJL156W-A YJL156W-A -2.25 unknown unknown YMR105C PGM2 -2.23 glucose 1-phosphate utilization phosphoglucomutase activity YGR248W SOL4 -2.00 unknown unknown YHR087W YHR087W -1.95 unknown unknown YPR160W GPH1 -1.76 glycogen catabolism glycogen phosphorylase activity YLR178C TFS1 -1.76 regulation of proteolysis lipid binding activity YOR173W DCS2 -1.72 unknown...”',
          'title': 'Microarray analysis of the in vivo sequence preferences of a minor groove binding drug',
          'journal': 'BMC genomics 2008',
          'authors': 'Eckdahl TT, Brown AD, Hart SN, Malloy KJ, Shott M, Yiu G, Hoopes LL, Heyer LJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1865649',
          'text': '“...S S Identification no. YBR005W YLR107W YPR108W YGR248W YLR136C YKL216W YOR230W YKL126W YDL124W YHL021C YKR049C YMR251W-A YGR268C YJL082W YML128C YLR350W YPR154W...”',
          'title': 'Mg2+ deprivation elicits rapid Ca2+ uptake and activates Ca2+/calcineurin signaling in Saccharomyces cerevisiae',
          'journal': 'Eukaryotic cell 2007',
          'authors': 'Wiesenberger G, Steinleitner K, Malli R, Graier WF, Vormann J, Schweyen RJ, Stadler JA.'}],
      'url': 'http://www.uniprot.org/uniprot/P53315',
      'db': 'SwissProt',
      'gene_id': 'P53315',
      'identity': 1.0,
      'coverage': 1.0},
    'SOL3_YEAST / P38858': {'organism': "Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast)",
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4057618',
        'text': '“...Q9BV23 P53750 O43175 P40054 P08243 P49090 P56937 Q12452 P22830 P16622 Q86V88 P40081 Q16769 P43599 O95336 P38858 Q8IXB1 P40564 Q15386 P53119 Q969P6 P04786 Q9Y3B8 P54964 Q9Y2H1 P53894 Q9Y2Z4 P48527 P00813 P53909 Q9UI42 P38836 Q9Y6R4 P53599 This table lists the pool of 113 reciprocal best match human-yeast ortholog...”',
        'title': 'A framework for application of metabolic modeling in yeast to predict the effects of nsSNV in human orthologs',
        'journal': 'Biology direct 2014',
        'authors': 'Dingerdissen H, Weaver DS, Karp PD, Pan Y, Simonyan V, Mazumder R.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4668057',
          'text': '“...were needed to produce any product. (PNG) Click here for additional data file. S8 Fig YHR163W, YPR073C, YGR125W, YJL134W. Confirmation gel for strains YHR163W, YPR073C, YGR125W, YJL134W after selection on FOA, with expected product sizes shown at bottom of lanes. (PNG) Click here for additional data...”',
          'title': 'A Tool for Multiple Targeted Genome Deletions that Is Precise, Scar-Free, and Suitable for Automation',
          'journal': 'PloS one 2015',
          'authors': 'Aubrey W, Riley MC, Young M, King RD, Oliver SG, Clare A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3618361',
          'text': '“...YGR001C YGR001c AGP3 YFL055w LEU3 YLR451w SNF6 YHL025w YGR031W YGR031w AIM23 YJL131c LIP5 YOR196c SOL3 YHR163w YGR051C YGR051c ALR2 YFL050c MET22 YOL064c SPE1 YKL184w YGR149W YGR149w APM4 YOL062c MON1 YGL124c SPS22 YCL048w YGR176W YGR176w APS1 YLR170c MPC54 YOR177c SRN2 YLR119w YGR207C YGR207c ARN2 YHL047c MRM2 YGL136c...”',
          'title': 'A Whole Genome Screen for Minisatellite Stability Genes in Stationary-Phase Yeast Cells',
          'journal': 'G3 (Bethesda, Md.) 2013',
          'authors': "Alver B, Jauert PA, Brosnan L, O'Hehir M, VanderSluis B, Myers CL, Kirkpatrick DT."},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163424',
          'text': '“...0.034 0.805 R_FTHFCLm YER183C 0.022 0 0.014 0 R_G6PDA YGR248W 0.017 0 0.007 0 R_G6PDA YHR163W 0.222 0 0.005 0 R_G6PDA YNR034W 0.023 0 0.028 0 R_GLUN YIL033C 0.079 0 0.205 0 R_M1PD YNR073C 0.016 0 0.024 0 R_MACACI YLL060C 0.011 0 0.014 0 R_POLYAO2 YMR020W...”',
          'title': 'On the formalization and reuse of scientific research',
          'journal': 'Journal of the Royal Society, Interface 2011',
          'authors': 'King RD, Liakata M, Lu C, Oliver SG, Soldatova LN.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1131933',
          'text': '“...YER050C RSM18 133(2) 297(1) 571(1) 5 +192 No YGR088W CTT1 8(2) 377(1) 7 +33 No YHR163W SOL3 91(2) 848(1) 8 +93 Yes YIL043C CBR1 82(1) 94(1) 102(3) 9 +114 Yes YIL053W RHR2 35(5) 36(1) 349(1) 526(1) 852(1) 9 +63 No YIL076W SEC28 177(4) 556(1) 9 +189...”',
          'title': "Mapping of transcription start sites in Saccharomyces cerevisiae using 5' SAGE",
          'journal': 'Nucleic acids research 2005',
          'authors': 'Zhang Z, Dietrich FS.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1175969',
          'text': '“...1 1 1 sol2 /YCRX13W 0.99 0.98 1 idp1 /YDL066W 0.92 0.94 1.03 sol3 / YHR163W 0.71 0.94 1 idp2 /YLR174W 0.86 0.96 0.95 sol4 / YGR248W 0.95 0.91 1.01 lsc1 /YOR142W 1.05 0.93 1 tal1 / YLR354C 0.89 0.94 1 mae1 /YKL029C 1.01 0.96 1...”',
          'title': 'Large-scale 13C-flux analysis reveals mechanistic principles of metabolic network robustness to null mutations in yeast',
          'journal': 'Genome biology 2005',
          'authors': 'Blank LM, Kuepfer L, Sauer U.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC383160',
          'text': '“...2.6 2.4 2.5 MEP3 YPR138C 2.9 PUT4 PYK2 SOL3 SUL2 YOR348C YOR347C YHR163W YLR092W YEL041W YER053C 6.6 12 2.6 2.2 4 2.7 5 9.1 2.2 2 3.5 2.6 3.4 3.1 2.6 2.1 26 2.5...”',
          'title': 'Molecular basis for anaerobic growth of Saccharomyces cerevisiae on xylose, investigated by global gene expression and metabolic flux analysis',
          'journal': 'Applied and environmental microbiology 2004',
          'authors': 'Sonderegger M, Jeppsson M, Hahn-Hägerdal B, Sauer U.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC524422',
          'text': '“...ENO1 RAD2 TN7-48D6 TN7-29G4 TN7-60C8 TN7-18E8 YHR141C YHR163W YIL123W YIL154C RPL42B SOL3 SIM1 IMP2 TN7-33E10 TN7-49C9 YJL117W YJR052W PHO86 RAD7 TN7-60F6...”',
          'title': 'Large-scale mutagenesis of the yeast genome using a Tn7-derived multipurpose transposon',
          'journal': 'Genome research 2004',
          'authors': 'Kumar A, Seringhaus M, Biery MC, Sarnovsky RJ, Umansky L, Piccirillo S, Heidtman M, Cheung KH, Dobry CJ, Gerstein MB, Craig NL, Snyder M.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC143595',
          'text': '“...YHR104W YLR070C YGR194C Pentose phosphate pathway YNL241C YHR163W HXT4 HXT5 HXT7 GAL2 High-affinity glucose transporter Hexose transporter Hexose transporter...”',
          'title': 'Molecular analysis of a Saccharomyces cerevisiae mutant with improved ability to utilize xylose shows enhanced expression of proteins involved in transport, initial xylose metabolism, and the pentose phosphate pathway',
          'journal': 'Applied and environmental microbiology 2003',
          'authors': 'Wahlbom CF, Cordero Otero RR, van Zyl WH, Hahn-Hägerdal B, Jönsson LJ.'}],
      'url': 'http://www.uniprot.org/uniprot/P38858',
      'db': 'SwissProt',
      'gene_id': 'P38858',
      'identity': 0.49,
      'coverage': 0.97},
    'PAS_chr3_1126': {'organism': 'Komagataella phaffii GS115',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4580311',
        'text': '“...ZWF1 PP7435_Chr2-0993/ PAS_chr2-1_0308 Glucose-6-phosphate dehydrogenase not changed not changed no no no WPVTRPDVLHKM SOL3 PP7435_Chr3-0037/ PAS_chr3_1126 6-phosphogluconolactonase not changed n.q. no no Twilight zone ALSGVSVSTSKY GND2 PP7435_Chr3-0944/ PAS_chr3_0277 6-phosphogluconate dehydrogenase not changed not changed yes yes no KGGNVSASTYDA Pentose phosphate pathway non-oxidative branch RPE1-1 PP7435_Chr3-0771 D-ribulose-5-phosphate...”',
        'title': 'Systems-level organization of yeast methylotrophic lifestyle',
        'journal': 'BMC biology 2015',
        'authors': 'Rußmayer H, Buchetics M, Gruber C, Valli M, Grillitsch K, Modarres G, Guerrasio R, Klavins K, Neubauer S, Drexler H, Steiger M, Troyer C, Al Chalabi A, Krebiehl G, Sonntag D, Zellnig G, Daum G, Graf AB, Altmann F, Koellensperger G, Hann S, Sauer M, Mattanovich D, Gasser B.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_002493372.1',
      'db': 'RefSeq',
      'gene_id': 'XP_002493372.1',
      'identity': 0.36,
      'coverage': 0.97},
    'SPCC16C4.10': {'organism': 'Schizosaccharomyces pombe',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5990653',
        'text': '“...LSM8 NO 7.805 0.01662 SPAC3H5.08c unknown human WDR44 downstream effector for RAB11 WDR44 7.764 0.02212 SPCC16C4.10 carbohydrate metabolic process PGLS 7.565 0.03079 msn5 nucleocytoplasmic transport XPO5 NO 7.185 0.02894 SPCC31H12.03c nucleocytoplasmic transport HNRNPUL1 7.077 0.0003582 hht1 chromatin organization HIST3H3 NO 6.83 0.001063 mug161 mRNA cis splicing...”',
        'title': 'The contribution of non-essential Schizosaccharomyces pombe genes to fitness in response to altered nutrient supply and target of rapamycin activity',
        'journal': 'Open biology 2018',
        'authors': 'Lie S, Banks P, Lawless C, Lydall D, Petersen J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4190384',
          'text': '“...K. lactis , the AER268W gene from A. gossypii has reported similarities in AGD to SPCC16C4.10 from Schizosaccharomyces pombe , which is associated to the referred enzyme. This phenotype was experimentally confirmed by a positive growth of A. gossypii on D-glucosamine (a product of chitin degradation)...”',
          'title': 'Genome-wide metabolic re-annotation of Ashbya gossypii: new insights into its metabolism through a comparative analysis with Saccharomyces cerevisiae and Kluyveromyces lactis',
          'journal': 'BMC genomics 2014',
          'authors': 'Gomes D, Aguiar TQ, Dias O, Ferreira EC, Domingues L, Rocha I.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3713068',
          'text': '“...oxPPP and trehalose metabolism pathways. Mutants lacking SPCC16C4.10 and SPCC794.10 were sensitive to cadmium. displayed large changes involving proteins in...”',
          'title': 'Cadmium-induced proteome remodeling regulated by Spc1/Sty1 and Zip1 in fission yeast',
          'journal': 'Toxicological sciences : an official journal of the Society of Toxicology 2012',
          'authors': 'Guo L, Ghassemian M, Komives EA, Russell P.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563147',
          'text': '“...S. cerevisiae ortholog SPAC1782.05 RRD2/YPL152W SPCC16C4.10 SPAC1805.07c SOL1/YNR034W None SPBC27.02c SPBC800.09 SPAC13C5.04 ASK1/YKL052C SCD6/YPR129W YLR126C...”',
          'title': 'A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe',
          'journal': 'Toxicological sciences : an official journal of the Society of Toxicology 2008',
          'authors': 'Kennedy PJ, Vashisht AA, Hoe KL, Kim DU, Park HO, Hayles J, Russell P.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=6577685',
      'db': 'MicrobesOnline',
      'gene_id': '6577685',
      'identity': 0.37,
      'coverage': 0.95},
    'AFLA_080390': {'organism': 'Aspergillus flavus NRRL3357',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6642104',
        'text': '“...pathway were up-regulated exposure to cinnamaldehyde including AFLA_041580 (estradiol 17 beta-dehydrogenase), AFLA_115890 (acyl-CoA oxidase) and AFLA_080390 (6-phosphogluconolactonase Sol). These results suggested that the down-regulation of fatty acids -oxidation and the up-regulation of pentose phosphate pathway were also associated with the anti-aflatoxigenic mechanism of cinnamaldehyde. Cinnamaldehyde was...”',
        'title': 'The anti-aflatoxigenic mechanism of cinnamaldehyde in Aspergillus flavus',
        'journal': 'Scientific reports 2019',
        'authors': 'Wang P, Ma L, Jin J, Zheng M, Pan L, Zhao Y, Sun X, Liu Y, Xing F.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6071284',
          'text': '“...down no zwf1 (AFLA_086620) 0.6412 0.4931 up no 0.7590 6.7 10 3 up no sol3 (AFLA_080390) 0.2110 0.8837 up no 1.0991 1.0 10 4 up yes gnd1 (AFLA_036840) 0.3718 0.7936 up no 0.9072 1.3 10 3 up no a : 0.2% ( w / v )...”',
          'title': 'The Antioxidant Gallic Acid Inhibits Aflatoxin Formation in Aspergillus flavus by Modulating Transcription Factors FarB and CreA',
          'journal': 'Toxins 2018',
          'authors': 'Zhao X, Zhi QQ, Li JY, Keller NP, He ZM.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_002372956.1',
      'db': 'RefSeq',
      'gene_id': 'XP_002372956.1',
      'identity': 0.39,
      'coverage': 0.92},
    'AOL_s00083g229': {'organism': 'Arthrobotrys oligospora ATCC 24927',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3164635',
        'text': '“...5. AOL_s00112g89; 6. AOL_s00004g627; 7. AOL_s00109g17; 8. AOL_s00110g144; 9. AOL_s00006g284; 10. AOL_s00078g394; 11. AOL_s00076g83; 12. AOL_s00083g229; 13. AOL_s00054g87; 14. AOL_s00004g628; 15. AOL_s00004g426; 16. AOL_s00054g899; 17. AOL_s00210g140; 18. AOL_s00112g112; 19. AOL_s00170g104; 20. AOL_s00043g45; 21. AOL_s00110g24; 22. AOL_s00075g130; 23. AOL_s00075g141; 24. AOL_s00004g494; 25. AOL_s00079g361; 26. AOL_s00215g818; 27. AOL_s00054g909;...”',
        'title': 'Genomic and proteomic analyses of the fungus Arthrobotrys oligospora provide insights into nematode-trap formation',
        'journal': 'PLoS pathogens 2011',
        'authors': 'Yang J, Wang L, Ji X, Feng Y, Li X, Zou C, Xu J, Ren Y, Mi Q, Wu J, Liu S, Liu Y, Huang X, Wang H, Niu X, Li J, Liang L, Luo Y, Ji K, Zhou W, Yu Z, Li G, Liu Y, Li L, Qiao M, Feng L, Zhang KQ.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_011123710.1',
      'db': 'RefSeq',
      'gene_id': 'XP_011123710.1',
      'identity': 0.38,
      'coverage': 0.95},
    'SOL2_YEAST / P37262': {'organism': "Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast)",
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5224485',
        'text': '“...name Mutation(s) suppressed by paralog 2 YBR181C PRS6B* cdc24, cdc15 YPL090C RPS6A* cdc24, cdc15, poc4 YCR073W-A SOL2 cdc26 YNR034W SOL1 cdc26 YDR099W BMH2* cdc25 YER177W BMH1* cdc25 YDR471W RPL27B* pol5 YHR010W RPL27A* pol5, cdc25 YER056C-A RPL34A* cdc48 YIL052C RPL34B* cdc48 YER131W RPS26B* vrp1 YGL189C RPL26A* vrp1...”',
        'title': 'A genome wide dosage suppressor network reveals genomic robustness',
        'journal': 'Nucleic acids research 2017',
        'authors': 'Patra B, Kon Y, Yadav G, Sevold AW, Frumkin JP, Vallabhajosyula RR, Hintze A, Østman B, Schossau J, Bhan A, Marzolf B, Tamashiro JK, Kaur A, Baliga NS, Grayhack EJ, Adami C, Galas DJ, Raval A, Phizicky EM, Ray A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4524479',
          'text': '“...or extensions, all within 50 bp from either end: YCL068C, YCL049C, YCL001W-B, YCR015C, YCR018C, YCR044C, YCR073W-A, YCR092C; multiple consecutive start codons in S. cerevisiae and only one in S. paradoxus : YCR038C, YCR042C, YCR073C). Finally, in the coding region of YCR028C there is polymorphism among European...”',
          'title': 'Estimating the Fitness Effects of New Mutations in the Wild Yeast Saccharomyces paradoxus',
          'journal': 'Genome biology and evolution 2015',
          'authors': 'Koufopanou V, Lomas S, Tsai IJ, Burt A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413506',
          'text': '“...frames which are present in Yeast 4 but not Yeast 5 are: YAL014C, YAL030W, YAR042W, YCR073W-A, YDL019C, YDR313C, YDR331W, YDR468C, YEL011W, YEL013W, YER093C, YFR055W, YGR199W, YHR005C, YHR073W, YMR068W, YNL006W, YNR034W, YOL078W, YPL145C, YPL275W, YPL276W, YOR237W, YIL105C, YJL058C, YJR160C, YKL203C, YML082W, and YNL047C. These open reading frames...”',
          'title': 'Yeast 5 - an expanded reconstruction of the Saccharomyces cerevisiae metabolic network',
          'journal': 'BMC systems biology 2012',
          'authors': 'Heavner BD, Smallbone K, Barker B, Mendes P, Walker LP.'}],
      'url': 'http://www.uniprot.org/uniprot/P37262',
      'db': 'SwissProt',
      'gene_id': 'P37262',
      'identity': 0.39,
      'coverage': 0.67},
    'FFUJ_02823': {'organism': 'Fusarium fujikuroi IMI 58289',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3694855',
        'text': '“...The gene codes of the 28 genes used for the calculation are: FFUJ_01340, FFUJ_01475, FFUJ_02452, FFUJ_02823, FFUJ_02928, FFUJ_02998, FFUJ_04170, FFUJ_04413, FFUJ_04421, FFUJ_04860, FFUJ_04893, FFUJ_05032, FFUJ_05937, FFUJ_06185, FFUJ_07582, FFUJ_08292, FFUJ_08315, FFUJ_08319, FFUJ_08541, FFUJ_08584, FFUJ_09575, FFUJ_09776, FFUJ_09852, FFUJ_13056, FFUJ_13472, FFUJ_13565, FFUJ_13760, FFUJ_13774. For phylogenetic analysis of SM genes,...”',
        'title': 'Deciphering the cryptic genome: genome-wide analyses of the rice pathogen Fusarium fujikuroi reveal complex regulation of secondary metabolism and novel metabolites',
        'journal': 'PLoS pathogens 2013',
        'authors': 'Wiemann P, Sieber CM, von Bargen KW, Studt L, Niehaus EM, Espino JJ, Huß K, Michielse CB, Albermann S, Wagner D, Bergner SV, Connolly LR, Fischer A, Reuter G, Kleigrewe K, Bald T, Wingfield BD, Ophir R, Freeman S, Hippler M, Smith KM, Brown DW, Proctor RH, Münsterkötter M, Freitag M, Humpf HU, Güldener U, Tudzynski B.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_023427926.1',
      'db': 'RefSeq',
      'gene_id': 'XP_023427926.1',
      'identity': 0.35,
      'coverage': 0.93},
    'SOL1_YEAST / P50278': {'organism': "Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast)",
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5224485',
        'text': '“...by paralog 2 YBR181C PRS6B* cdc24, cdc15 YPL090C RPS6A* cdc24, cdc15, poc4 YCR073W-A SOL2 cdc26 YNR034W SOL1 cdc26 YDR099W BMH2* cdc25 YER177W BMH1* cdc25 YDR471W RPL27B* pol5 YHR010W RPL27A* pol5, cdc25 YER056C-A RPL34A* cdc48 YIL052C RPL34B* cdc48 YER131W RPS26B* vrp1 YGL189C RPL26A* vrp1 YFR023W PES4 smc2...”',
        'title': 'A genome wide dosage suppressor network reveals genomic robustness',
        'journal': 'Nucleic acids research 2017',
        'authors': 'Patra B, Kon Y, Yadav G, Sevold AW, Frumkin JP, Vallabhajosyula RR, Hintze A, Østman B, Schossau J, Bhan A, Marzolf B, Tamashiro JK, Kaur A, Baliga NS, Grayhack EJ, Adami C, Galas DJ, Raval A, Phizicky EM, Ray A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086634',
          'text': '“...Translational elongation factor EF-3 YML087C AIM33 6.2 1.01 10 4 Putative protein of unknown function YNR034W SOL1 6.2 7.19 10 7 Protein with a possible role in tRNA export YDR070C FMP16 6.1 3.24 10 4 Protein of unknown function YJR129C EFM3 6.1 4.06 10 2 S-adenosylmethionine-dependent...”',
          'title': 'Different Toxicity Mechanisms for Citrinin and Ochratoxin A Revealed by Transcriptomic Analysis in Yeast',
          'journal': 'Toxins 2016',
          'authors': 'Vanacloig-Pedros E, Proft M, Pascual-Ahuir A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413506',
          'text': '“...YAR042W, YCR073W-A, YDL019C, YDR313C, YDR331W, YDR468C, YEL011W, YEL013W, YER093C, YFR055W, YGR199W, YHR005C, YHR073W, YMR068W, YNL006W, YNR034W, YOL078W, YPL145C, YPL275W, YPL276W, YOR237W, YIL105C, YJL058C, YJR160C, YKL203C, YML082W, and YNL047C. These open reading frames were removed from reaction annotations because of inadequate literature evidence supporting the Yeast 4...”',
          'title': 'Yeast 5 - an expanded reconstruction of the Saccharomyces cerevisiae metabolic network',
          'journal': 'BMC systems biology 2012',
          'authors': 'Heavner BD, Smallbone K, Barker B, Mendes P, Walker LP.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3148214',
          'text': '“...Cryptosporidium . TBLASTN searches using both Zwf1p and Sol1p ( S. cerevisiae 6PGL homolog, NCBI: NP_014432), as well as the homologous proteins identified in other apicomplexans, were performed against the Cryptosporidium genome sequences to verify that the gene was not missed during gene model annotation. Eimeria...”',
          'title': 'Multiple independent fusions of glucose-6-phosphate dehydrogenase with enzymes in the pentose phosphate pathway',
          'journal': 'PloS one 2011',
          'authors': 'Stover NA, Dixon TA, Cavalcanti AR.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163424',
          'text': '“...0.014 0 R_G6PDA YGR248W 0.017 0 0.007 0 R_G6PDA YHR163W 0.222 0 0.005 0 R_G6PDA YNR034W 0.023 0 0.028 0 R_GLUN YIL033C 0.079 0 0.205 0 R_M1PD YNR073C 0.016 0 0.024 0 R_MACACI YLL060C 0.011 0 0.014 0 R_POLYAO2 YMR020W 0.016 0 0.023 0 R_PUNP1 YLR017W...”',
          'title': 'On the formalization and reuse of scientific research',
          'journal': 'Journal of the Royal Society, Interface 2011',
          'authors': 'King RD, Liakata M, Lu C, Oliver SG, Soldatova LN.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2645918',
          'text': '“...completion of separation 11 YNL098C RAS2 sporulation 12 YGR162W TIF4631 ribosome biogenesis and assembly 13 YNR034W SOL1 tRNA export from nucleus 14 YER044C ERG28 ergosterol biosynthesis 15 YGR078C PAC10 tubulin folding 16 YGL249W ZIP2 synapsis 17 YIR033W MGA2 regulation of transcription from polymerase II promoter 18...”',
          'title': "Identification of human intracellular targets of the medicinal Herb St. John's Wort by chemical-genetic profiling in yeast",
          'journal': 'Journal of agricultural and food chemistry 2008',
          'authors': 'McCue PP, Phang JM.'}],
      'url': 'http://www.uniprot.org/uniprot/P50278',
      'db': 'SwissProt',
      'gene_id': 'P50278',
      'identity': 0.36,
      'coverage': 0.66},
    '6PGL_RAT / P85971': {'organism': 'Rattus norvegicus (Rat)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3761203',
        'text': '“...3.73 SpC SpC P98158 P04176 D4ACL3 D3ZPA9 Q5RKI0 P85971 LRP2_RAT PH4H_RAT D4ACL3_RAT D3ZPA9_RAT WDR1_RAT 6PGL_RAT Lrp2 Pah Aco1 Ctsa Wdr1 Pgls 0.008 0.012 0.013...”',
        'title': 'Proteomic profiling and pathway analysis of the response of rat renal proximal convoluted tubules to metabolic acidosis',
        'journal': 'American journal of physiology. Renal physiology 2013',
        'authors': 'Schauer KL, Freund DM, Prenni JE, Curthoys NP.'}],
      'url': 'http://www.uniprot.org/uniprot/P85971',
      'db': 'SwissProt',
      'gene_id': 'P85971',
      'identity': 0.34,
      'coverage': 0.95},
    'PAAG_05621': {'organism': 'Paracoccidioides lutzii Pb01',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471430',
        'text': '“...PAAG_06796 711.90 Associate energy conservation Cytochrome c PriAC=F2TJX0 PAAG_06268 1307.21 1.522 Glycolysis and gluconeogenesis 6-phosphogluconolactonase PAAG_05621 688.50 1.297 Glyoxylate cycle Malate synthase PAAG_04542 617.40 * Krebs cycle Succinyl-CoA: PAAG_05093 770.38 * Methyl citrate cycle 2-methylcitrate dehydratase PAAG_04559 20407.15 1.297 Oxidation of fatty acids Enoyl-CoA hydratase PAAG_06309...”',
        'title': 'Proteomic profile response of Paracoccidioides lutzii to the antifungal argentilactone',
        'journal': 'Frontiers in microbiology 2015',
        'authors': 'Prado RS, Bailão AM, Silva LC, de Oliveira CM, Marques MF, Silva LP, Silveira-Lacerda EP, Lima AP, Soares CM, Pereira M.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4014450',
          'text': '“...PAAG_02512 pyruvate decarboxylase # alcohol fermentation PAAG_08248 alcohol dehydrogenase # alcohol fermentation Pentose Phosphate pathway PAAG_05621 6-phosphogluconolactonase # pentose-phosphate shunt PAAG_05146 ribose 5-phosphate isomerase A # pentose-phosphate shunt CELL CYCLE and DNA PROCESSING PAAG_07608 DNA helicase # DNA repair PAAG_08917 histone H2a # DNA processing PAAG_07098...”',
          'title': 'Transcriptional and proteomic responses to carbon starvation in Paracoccidioides',
          'journal': 'PLoS neglected tropical diseases 2014',
          'authors': 'Lima Pde S, Casaletti L, Bailão AM, de Vasconcelos AT, Fernandes Gda R, Soares CM.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_002792336.1',
      'db': 'RefSeq',
      'gene_id': 'XP_002792336.1',
      'identity': 0.34,
      'coverage': 0.93},
    'CNAG_02133': {'organism': 'Cryptococcus neoformans var. grubii H99',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5959073',
        'text': '“...7.8 1.7E-11 Pentose-phosphate shunt (p-value : 5 . 3E-04) CNAG_03245 glucose-6-phosphate deHase 0.6 2.8 3.2E-04 CNAG_02133 6-phosphogluconolactonase 0.6 2.6 8.7E-04 CNAG_01492 hypothetical 6-phosphogluconate deHase 0.5 2.7 1.0E-03 CNAG_04099 6-phosphogluconate deHase, decarboxylating 1.2 1.2E-02 3.3 3.8E-05 CNAG_07561 6-phosphogluconate deHase, decarboxylating 1 0.5 2.3 8.9E-03 Others CNAG_06313 phosphoglucomutase...”',
        'title': 'Cryptococcus neoformans can form titan-like cells in vitro in response to multiple signals',
        'journal': 'PLoS pathogens 2018',
        'authors': 'Trevijano-Contador N, de Oliveira HC, García-Rodas R, Rossi SA, Llorente I, Zaballos Á, Janbon G, Ariño J, Zaragoza Ó.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3524413',
          'text': '“...TAGTGCCCG 3 16 CNAG_07561 6-phosphogluconate dehydrogenase AAGGTTGATG 1 14 CNAG_03146 alpha amylase GCACGGAGAT 5 13 CNAG_02133 6-phosphogluconolactonase CATTACTGCA 3 13 CNAG_01102 Glucose 1-dehydrogenase ATTCAAAAAA 1 13 CNAG_04640 ATP-citrate lyase, ACL1 ATAAGCTTTC 0 12 CNAG_07745 Mannitol-1-phosphate dehydrogenase GTCGTAGAGT 2 9 CNAG_03072 Enolase 3 TGTCAAAAAA 0 7 Transaldolase...”',
          'title': 'A defect in ATP-citrate lyase links acetyl-CoA production, virulence factor elaboration and virulence in Cryptococcus neoformans',
          'journal': 'Molecular microbiology 2012',
          'authors': 'Griffiths EJ, Hu G, Fries B, Caza M, Wang J, Gsponer J, Gates-Hollingsworth MA, Kozel TR, De Repentigny L, Kronstad JW.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_012049968.1',
      'db': 'RefSeq',
      'gene_id': 'XP_012049968.1',
      'identity': 0.35,
      'coverage': 0.8},
    'PADG_07771': {'organism': 'Paracoccidioides brasiliensis Pb18',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6371752',
        'text': '“...are ETC and ATP synthesis. TCA, tricarboxylic cicle acid; ETC, electron transport chain; PADG_04604, transketolase; PADG_07771, 6-phosphogluconolactonase; PADG_03651, 6-phosphogluconate dehydrogenase, decarboxylating 1; PADG_07420, transaldolase; PADG_08387, citrate synthase mitochondrial; PADG_11845, aconitate hydratase mitochondrial; PADG_00317, succinyl-CoA ligase subunit beta; PADG_04939, succinyl-CoA, 3-ketoacid-coenzyme A transferase; PADG_01762, oxoglutarate dehydrogenase; PADG_00052,...”',
        'title': 'Proteomic Analysis of Paracoccidioides brasiliensis During Infection of Alveolar Macrophages Primed or Not by Interferon-Gamma',
        'journal': 'Frontiers in microbiology 2019',
        'authors': 'Chaves EGA, Parente-Rocha JA, Baeza LC, Araújo DS, Borges CL, de Oliveira MAP, Soares CMA.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711425',
          'text': '“...synthase (PADG_04702), ECH enoyl-CoA hydratase (PADG_01209), ACD acyl-CoA dehydrogenase (PADG_07604) PLA phospholipase (PADG_05993), PGL phosphoglucolactonase (PADG_07771), TRX thioredoxins (PADG_03161 and PADG_05504), TrxR thioredoxin reductase (PADG_01551), SOD superoxide dismutase (PADG_07418 and PADG_01755), CCP cytochrome C peroxidase (PADG_03163). The MultiExperiment Viewer software V.4.8 ( www.tm4.org/mev/ ) was used...”',
          'title': 'Paracoccidioides brasiliensis presents metabolic reprogramming and secretes a serine proteinase during murine infection',
          'journal': 'Virulence 2017',
          'authors': 'Lacerda Pigosso L, Baeza LC, Vieira Tomazett M, Batista Rodrigues Faleiro M, Brianezi Dignani de Moura VM, Melo Bailão A, Borges CL, Alves Parente Rocha J, Rocha Fernandes G, Gauthier GM, Soares CMA.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_010763145.1',
      'db': 'RefSeq',
      'gene_id': 'XP_010763145.1',
      'identity': 0.35,
      'coverage': 0.93},
    'O95336': {'organism': 'Homo sapiens',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6414445',
        'text': '“...1.1.1.27 0.27 0.38 Pentose Phosphate Pathway Glucose-6-phosphate dehydrogenase (G6PD) P11413 1.1.149/1.1.1.343 0.22 0.29 6-phosphogluconolactonase (PGLS) O95336 3.1.1.31 0.80 0.52 Phosphogluconate dehydrogenase (PGD) P52209 1.1.1.44/1.1.1.343 0.48 0.25 Glucose-6-phosphate isomerase (GPI) P06744 5.3.1.9 0.41 0.40 Transketolase (TKT) P29401 2.2.1.1 0.64 0.60 Transaldolase 1 (TALDO1) P37837 2.2.1.2 0.57 0.43...”',
        'title': 'A Pathogen and a Non-pathogen Spotted Fever Group Rickettsia Trigger Differential Proteome Signatures in Macrophages',
        'journal': 'Frontiers in cellular and infection microbiology 2019',
        'authors': 'Curto P, Santa C, Allen P, Manadas B, Simões I, Martinez JJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5907394',
          'text': '“...272 6 5 20.6 10 Glucose-6-phosphate 1-dehydrogenase P11413 G6PD 45 1 1 11.8 11 6-Phosphogluconolactonase O95336 6PGL Western blot 12 6-Phosphogluconate dehydrogenase, decarboxylating P52209 PGD 162 3 3 22.2 160 3 3 24.2 13 Transketolase P29401 TKT 238 7 4 14.4 185 6 4 19.3 14...”',
          'title': 'Young and Especially Senescent Endothelial Microvesicles Produce NADPH: The Fuel for Their Antioxidant Machinery',
          'journal': 'Oxidative medicine and cellular longevity 2018',
          'authors': 'Bodega G, Alique M, Bohórquez L, Morán M, Magro L, Puebla L, Ciordia S, Mena MC, Arza E, Ramírez MR.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5431125',
          'text': '“...Gamma-synuclein O76070 0.39 0.28 0.71 67 Actin-related protein 2 P61160 2.01 1.42 0.71 68 6-phosphogluconolactonase O95336 2.21 1.56 0.70 69 Keratin, type I cytoskeletal 10 P13645 1.20 0.83 0.70 70 Heat shock protein beta-1 P04792 2.02 1.40 0.69 71 Peroxiredoxin-6 P30041 1.49 1.02 0.69 72 Trifunctional...”',
          'title': 'Subventricular zone involvement in Glioblastoma - A proteomic evaluation and clinicoradiological correlation',
          'journal': 'Scientific reports 2017',
          'authors': 'Gollapalli K, Ghantasala S, Kumar S, Srivastava R, Rapole S, Moiyadi A, Epari S, Srivastava S.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5548489',
          'text': '“...homolog OS= Homo sapiens GN=PUS7 PE=1 SV=2 661 75186.30 20 14 26.30 1.00 7.98 10.99 O95336 6 PGL_HUMAN 6-phosphogluconolactonase OS= Homo sapiens GN=PGLS PE=1 SV=2 258 27601.60 13 8 41.50 0.99 6.00 6.04 Q8TD06 AGR3_HUMAN Anterior gradient protein three homolog OS= Homo sapiens GN=AGR3 PE=1 SV=1...”',
          'title': 'Estrogen receptor coregulator binding modulators (ERXs) effectively target estrogen receptor positive human breast cancers',
          'journal': 'eLife 2017',
          'authors': 'Raj GV, Sareddy GR, Ma S, Lee TK, Viswanadhapalli S, Li R, Liu X, Murakami S, Chen CC, Lee WR, Mann M, Krishnan SR, Manandhar B, Gonugunta VK, Strand D, Tekmal RR, Ahn JM, Vadlamudi RK.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5382314',
          'text': '“...1.3 16 P00367 Glutamate dehydrogenase 1, mitochondrial 1.4 25 P11172 Uridine 5-monophosphate synthase 0.7 31 O95336 6-Phosphogluconolactonase 1.3 Inflammatory response 39a-b P06702 Protein S100 A9 0.7 42a-b P05109 Protein S100 A8 0.6 26 P04083 Annexin A1 0.6 Signal transduction 27 P63104 14-3-3 protein zeta/delta 1.3 28a...”',
          'title': 'Decreased Levels of Foldase and Chaperone Proteins Are Associated with an Early-Onset Amyotrophic Lateral Sclerosis',
          'journal': 'Frontiers in molecular neuroscience 2017',
          'authors': 'Filareti M, Luotti S, Pasetto L, Pignataro M, Paolella K, Messina P, Pupillo E, Filosto M, Lunetta C, Mandrioli J, Fuda G, Calvo A, Chiò A, Corbo M, Bendotti C, Beghi E, Bonetto V.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4773818',
          'text': '“...dehydrogenase 16 5.338** 108 751 30.1 P07384 Calpain-1 catalytic subunit 19 7.247* 109 564 62 O95336 6-phosphogluconolactonase 11 11.246** 110 176 14.1 Q9H9T3-2 Isoform 2 of Elongator complex protein 3 6 5.245* 111 2382 35.4 Q53F91 Villin 1 variant 27 3.245** 112 247 31.4 B4DP27 cDNA...”',
          'title': 'Pharmacoproteomic analysis reveals that metapristone (RU486 metabolite) intervenes E-cadherin and vimentin to realize cancer metastasis chemoprevention',
          'journal': 'Scientific reports 2016',
          'authors': 'Yu S, Yan C, Yang X, He S, Liu J, Qin C, Huang C, Lu Y, Tian Z, Jia L.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5090863',
          'text': '“...exosome (extracellular) na hydrolase glucose metabolisme Glycogen storage disease 10 (lysosomal storage disease) Y/Y PGLS O95336 6-phosphogluconolactonase 1133 P cytoplasma na hydrolase carbohydrate metabolisme Diabetic nephropathy Y/Y ALDOB P05062 Fructose-bisphosphate aldolase B 1327 P cytoplasma na lyase glycolysis Hereditary fructose intolerance Y/Y DCXR Q7Z4W1 L-xylulose reductase...”',
          'title': 'Isolation and mass spectrometry analysis of urinary extraexosomal proteins',
          'journal': 'Scientific reports 2016',
          'authors': 'Hildonen S, Skarpen E, Halvorsen TG, Reubsaet L.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3796057',
          'text': '“...Q96DA0 Zymogen granule protein 16 homolog B 6.2E+02 Ad 1.5 P00450 Ceruloplasmin 3.9E+02 Ad 1.5 O95336 6-phosphogluconolactonase 6.9E+01 Ad 1.5 P05062 Fructose-bisphosphate aldolase B 3.2E+01 Ad 1.5 Q8WZ75 Roundabout homolog 4 5.5E+02 Ad 1.5 Q14393 Growth arrest-specific protein 6 3.0E+01 Ad 1.5 O00468 Agrin 6.3E+02 Ad...”',
          'title': 'An in-depth comparison of the male pediatric and adult urinary proteomes',
          'journal': 'Biochimica et biophysica acta 2014',
          'authors': 'Froehlich JW, Vaezzadeh AR, Kirchner M, Briscoe AC, Hofmann O, Hide W, Steen H, Lee RS.'}],
      'url': 'http://www.brenda-enzymes.org/sequences.php?AC=O95336',
      'db': 'BRENDA',
      'gene_id': 'O95336',
      'identity': 0.34,
      'coverage': 0.95},
    'XP_003194407': {'organism': 'Cryptococcus gattii WM276',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4132117',
        'text': '“...hydratase 48 AFR93622 5 16 7 Phosphopyruvate hydratase 48 AFR97782 12 32 8 6-phosphogluconolactonase 40 XP_003194407 8 21 9 Mitochondrial precursor 32 XP_003194323 4 16 10 Hypothetical protein CGB_M1040C 15 XP_003197291 4 36 11 Hypothetical protein CGB_M1040C 15 XP_003197291 8 48 12 Translation initiation factor 5a...”',
        'title': 'Vaccine-mediated immune responses to experimental pulmonary Cryptococcus gattii infection in mice',
        'journal': 'PloS one 2014',
        'authors': 'Chaturvedi AK, Hameed RS, Wozniak KL, Hole CR, Leopold Wager CM, Weintraub ST, Lopez-Ribot JL, Wormley FL.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_003194407.1',
      'db': 'RefSeq',
      'gene_id': 'XP_003194407.1',
      'identity': 0.36,
      'coverage': 0.7},
    'CG17333': {'organism': 'Drosophila melanogaster',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4869932',
        'text': '“...mutants, along with phosphoglucomutase (pgm ), which is involved in glycogen metabolism, and transaldolase and CG17333 , which are involved in the pentose phosphate shunt. Additionally, the gluconeogenesis genes Pyruvate carboxylase (CG1514 ) and Phosphoenolpyruvate carboxykinase (Pepck, CG17725 ) show reduced expression in mutant animals, similar...”',
        'title': 'The Drosophila HNF4 nuclear receptor promotes glucose-stimulated insulin secretion and mitochondrial function in adults',
        'journal': 'eLife 2016',
        'authors': 'Barry WE, Thummel CS.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3352090',
          'text': '“...FBgn0023477 // CG2827 (1.27) transaldolase rep FBgn0004654 // CG3724 (2.15) phosphogluconate dehydrogenase rep FBgn0030239 // CG17333 (1.34) 6-phosphogluconolactonase pred FBgn0037607 // CG8036 (1.39) transketolase pred GO:0006769 nicotinamide metabolic process same five genes 64.4 GO:0019362 pyridine nucleotide metabolic process same five genes 60.1 GO:0006006 glucose metabolic process...”',
          'title': 'Drosophila poly suggests a novel role for the Elongator complex in insulin receptor-target of rapamycin signalling',
          'journal': 'Open biology 2012',
          'authors': 'Bolukbasi E, Vass S, Cobbe N, Nelson B, Simossis V, Dunbar DR, Heck MM.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/NP_572656.1',
      'db': 'RefSeq',
      'gene_id': 'NP_572656.1',
      'identity': 0.33,
      'coverage': 0.96},
    'Synpcc7942_0529': {'organism': 'Synechococcus elongatus PCC 7942',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4672817',
        'text': '“...pathway encode 6-phosphogluconolactonase (pgl; SynPCC7942_0529), glucose6-phosphate 1-dehydrogenase (zwf; SynPCC7942_2334), and 6-phosphogluconate dehydrogenase...”',
        'title': 'The essential gene set of a photosynthetic organism',
        'journal': 'Proceedings of the National Academy of Sciences of the United States of America 2015',
        'authors': 'Rubin BE, Wetmore KM, Price MN, Diamond S, Shultzaberger RK, Lowe LC, Curtin G, Arkin AP, Deutschbauer A, Golden SS.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=1174062',
      'db': 'MicrobesOnline',
      'gene_id': '1174062',
      'identity': 0.32,
      'coverage': 0.96},
    'GRMZM2G148769, umc2374': {'organism': 'Zea mays',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4668103',
        'text': '“...2:221504148221501546 3.20E-88 256 Y/N A XZ-D 12 gi|224029461 unknown [Zea mays] 28955.8/5.08 201 100 12 GRMZM2G148769 2:221504148221501546 3.20E-88 276 Y/N + LZ-D 12 gi|224029461 unknown [Zea mays] 28955.8/5.08 238 100 9 GRMZM2G148769 2:221504148221501546 3.20E-88 167 2.6 + XZ-D 10 gi|194697638 unknown [Zea mays] 64089/4.8 270 100...”',
        'title': 'Proteomic Analysis of Silk Viability in Maize Inbred Lines and Their Corresponding Hybrids',
        'journal': 'PloS one 2015',
        'authors': 'Ma Z, Qin Y, Wang Y, Zhao X, Zhang F, Tang J, Fu Z.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3691554',
          'text': '“...to alpha-galactosidase from Oryza sativa (Q9FXT4) 1.42 0.09 EC 3.2.1.22 zma00052, zma0056, zma00600, zma00603 B6UAK0 umc2374; LOC100285843 (EC:3.1.1.31); K01057 6-phosphogluconolactonase [EC:3.1.1.31] 1.49 0.3 zma:100285843 zma00030, zma01100, zma01110 B4G1X1 Uncharacterized LOC100274554; K07407 alpha-galactosidase [EC:3.2.1.22] 1.59 0.05 zma:100274554 zma00561, zma00600, zma00603 KEGG pathway identifiers: zma00010:glycolysis/gluconeogenesis; zma00020:citrate cycle (TCA...”',
          'title': 'Proteomic comparison of basal endosperm in maize miniature1 mutant and its wild-type Mn1',
          'journal': 'Frontiers in plant science 2013',
          'authors': 'Silva-Sanchez C, Chen S, Zhu N, Li QB, Chourey PS.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/NP_001152205.1',
      'db': 'RefSeq',
      'gene_id': 'NP_001152205.1',
      'identity': 0.33,
      'coverage': 0.77},
    'Bm1_33900': {'organism': 'Brugia malayi',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3501679',
        'text': '“...hypothetical protein AF, EE, MF, L3 unknown Bm1_32410 1841 4-hydroxyphenylpyruvate dioxygenase family protein Constitutive metabolism Bm1_33900 659 6 phosphogluconolactonase family protein Constitutive metabolism Bm1_38530 1083 PET Domain containing protein AF, EE, MF signalling Bm1_42085 2818 Tristetraproline AF, EE, MF, L3 transcription Bm1_42090 1198 MGC79044 protein-related AM...”',
        'title': 'Identification of genes containing ecdysone response elements in the genome of Brugia malayi',
        'journal': 'Molecular and biochemical parasitology 2012',
        'authors': 'Liu C, Enright T, Tzertzinis G, Unnasch TR.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_001898227.1',
      'db': 'RefSeq',
      'gene_id': 'XP_001898227.1',
      'identity': 0.32,
      'coverage': 0.92},
    'CNAG_01216': {'organism': 'Cryptococcus neoformans var. grubii H99',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6867901',
        'text': '“...CNAG_07561 6-Phosphogluconate dehydrogenase, decarboxylating I 5.3 18.8 CNAG_06175 26S proteasome, regulatory subunit N2 4.7 0.0 CNAG_01216 6-Phosphogluconolactonase 4.0 0.0 CNAG_00602 Eukaryotic translation initiation factor 3, subunit I 4.0 0.0 CNAG_00136 Ubiquitin-activating enzyme E1 4.0 0.0 CNAG_06666 -1,4-Glucan phosphorylase 3.7 0.0 CNAG_02565 Homoaconitase, mitochondrial 3.7 0.0 CNAG_02035...”',
        'title': 'A Fungal Arrestin Protein Contributes to Cell Cycle Progression and Pathogenesis',
        'journal': 'mBio 2019',
        'authors': 'Telzrow CL, Nichols CB, Castro-Lopez N, Wormley FL, Alspaugh JA.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_012049395.1',
      'db': 'RefSeq',
      'gene_id': 'XP_012049395.1',
      'identity': 0.3,
      'coverage': 0.83},
    'NP_775547': {'organism': 'Mus musculus',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105561',
        'text': 'GeneRIF: Studied expression levels and signal pathways involved with hexose-6-phosphate dehydrogenase in mouse model of diet induced visceral adiposity.',
        'title': 'Enhanced hexose-6-phosphate dehydrogenase expression in adipose tissue may contribute to diet-induced visceral adiposity.',
        'journal': 'International journal of obesity (2005) 2018',
        'authors': 'Liu L, Wang Y, Wang J, Dong Y, Chang S, Liu X, Lutfy K, Chen H, Friedman TC, Jiang M, Liu Y'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5601182',
          'text': 'GeneRIF: We envisage these data and models finding utility when investigating the muscle-specific functions of the 11beta-HSD1/G6PT/H6PDH triad.',
          'title': 'Cellular and genetic models of H6PDH and 11β-HSD1 function in skeletal muscle.',
          'journal': 'Cell biochemistry and function 2017',
          'authors': 'Zielinska AE, Fletcher RS, Sherlock M, Doig CL, Lavery GG'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3948972',
          'text': 'GeneRIF: These results suggest that increased H6PDH expression specifically in adipose tissue is sufficient to cause intra-adipose glucocorticoid production and adverse metabolic phenotypes.',
          'title': 'Transgenic overexpression of hexose-6-phosphate dehydrogenase in adipose tissue causes local glucocorticoid amplification and lipolysis in male mice.',
          'journal': 'American journal of physiology. Endocrinology and metabolism 2014',
          'authors': 'Wang Y, Liu L, Du H, Nagaoka Y, Fan W, Lutfy K, Friedman TC, Jiang M, Liu Y'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795617',
          'text': 'GeneRIF: Findings suggest that increased hepatic H6PDH and G6PT production contribute to 11beta-HSD1 upregulation of local glucocorticoid action that may be related to the development of type 2 diabetes.',
          'title': 'Tissue-specific dysregulation of hexose-6-phosphate dehydrogenase and glucose-6-phosphate transporter production in db/db mice as a model of type 2 diabetes.',
          'journal': 'Diabetologia 2011',
          'authors': 'Wang Y, Nakagawa Y, Liu L, Wang W, Ren X, Anghel A, Lutfy K, Friedman TC, Liu Y'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219053',
          'text': 'GeneRIF: These data are consistent with an 11beta-HSD1-independent function for H6PDH in which sarcoplasmic reticulum G6P metabolism and NAD/NADH redox status are important for maintaining muscle homeostasis.',
          'title': 'Hexose-6-phosphate dehydrogenase contributes to skeletal muscle homeostasis independent of 11β-hydroxysteroid dehydrogenase type 1.',
          'journal': 'Endocrinology 2011',
          'authors': 'Semjonous NM, Sherlock M, Jeyasuria P, Parker KL, Walker EA, Stewart PM, Lavery GG'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879949',
          'text': 'GeneRIF: High-glucose-mediated decrease in G6PD activity may provide a mechanistic explanation for the gradual loss of beta cells in patients with diabetes.',
          'title': 'High glucose inhibits glucose-6-phosphate dehydrogenase, leading to increased oxidative stress and beta-cell apoptosis.',
          'journal': 'FASEB journal : official publication of the Federation of American Societies for Experimental Biology 2010',
          'authors': 'Zhang Z, Liew CW, Handy DE, Zhang Y, Leopold JA, Hu J, Guo L, Kulkarni RN, Loscalzo J, Stanton RC'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2329282',
          'text': 'GeneRIF: Lack of hexose-6-phosphate dehydrogenase impairs lipid mobilization from mouse adipose tissue.',
          'title': 'Lack of hexose-6-phosphate dehydrogenase impairs lipid mobilization from mouse adipose tissue.',
          'journal': 'Endocrinology 2008',
          'authors': 'Bujalska IJ, Hewitt KN, Hauton D, Lavery GG, Tomlinson JW, Walker EA, Stewart PM'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/NP_775547.2',
      'db': 'RefSeq',
      'gene_id': 'NP_775547.2',
      'identity': 0.32,
      'coverage': 0.26},
    'XP_006239521': {'organism': 'Rattus norvegicus',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3741409',
        'text': 'GeneRIF: Activation of expression and activity of Hexose-6-phosphate dehydrogenase in primary hepatocytes was mediated through levels of glucose, not related to insulin levels so concluding that glucose instead of insulin directly regulates H6PDH.',
        'title': 'Direct regulation of glucose and not insulin on hepatic hexose-6-phosphate dehydrogenase and 11β-hydroxysteroid dehydrogenase type 1.',
        'journal': 'Molecular and cellular endocrinology 2011',
        'authors': 'Fan Z, Du H, Zhang M, Meng Z, Chen L, Liu Y'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2219311',
          'text': 'GeneRIF: H6PDH was amply expressed in most tissues examined in which 11beta-HSD1 was also expressed, with the notable exception of the renal interstitial cells',
          'title': 'Hexose-6-phosphate dehydrogenase and 11beta-hydroxysteroid dehydrogenase-1 tissue distribution in the rat.',
          'journal': 'Endocrinology 2008',
          'authors': 'Gomez-Sanchez EP, Romero DG, de Rodriguez AF, Warden MP, Krozowski Z, Gomez-Sanchez CE'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_006239521.1',
      'db': 'RefSeq',
      'gene_id': 'XP_006239521.1',
      'identity': 0.32,
      'coverage': 0.27},
    '6PGL4_ARATH / Q9FIN1': {'organism': 'Arabidopsis thaliana (Mouse-ear cress)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3142401',
        'text': '“...-0.39 0.37 AT3G49360 0.39 -0.51 AT5G24420 0.31 -0.33 AT5G24410 -0.33 0.40 AT3G04790 0.84 -0.70 AT5G44520 0.70 -0.77 AT2G01290 0.68 -0.65 AT1G71100 -0.11 0.24...”',
        'title': 'Divergent evolutionary fates of major photosynthetic gene networks following gene and whole genome duplications',
        'journal': 'Plant signaling & behavior 2011',
        'authors': 'Coate JE, Doyle JJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1621065',
          'text': '“...decreased (Table 3 ), including D-ribulose-5-phosphate 3-epimerase (At3g01850, At1g63290), ribose 5-phosphate isomerase (At2g01290, At3g04790), 6-phosphogluconolactonase (At5g24410, At3g02360), and glucose-6-phosphate-1-dehydrogenase (G6PDH) ( Additional File 1 ). The PPP is an important source of NADPH, especially in non-photosynthetic tissues, and therefore might be expected to have increased activity...”',
          'title': 'Comprehensive transcriptional profiling of NaCl-stressed Arabidopsis roots reveals novel classes of responsive genes',
          'journal': 'BMC plant biology 2006',
          'authors': 'Jiang Y, Deyholos MK.'}],
      'url': 'http://www.uniprot.org/uniprot/Q9FIN1',
      'db': 'SwissProt',
      'gene_id': 'Q9FIN1',
      'identity': 0.32,
      'coverage': 0.83},
    '6PGL3_ARATH / Q84WW2': {'organism': 'Arabidopsis thaliana (Mouse-ear cress)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6660741',
        'text': '“...Eco-type Position At5g17630 XPT Xul5P/phosphate translocator 2 xpt-2 SAIL_378_C01 Col +68 xpt-4 GT_5_112515 Ler +1017 At5g24400 PGL3 6-Phosphogluconolactonase 6 pgl3-1 SALK_005685 Col +975 pgl3-2 FLAG_219G10 WS +250 pgl3-3 emb2024 Col +215/437 At1g64190 PGD1 6-Phosphogluconate dehydrogenase 7 pgd1-1 GABI_762C02 Col +11 pgd1-2 SALK_121521 Col +1445 NA SALK_002712...”',
        'title': 'The plastidial pentose phosphate pathway is essential for postglobular embryo development in Arabidopsis',
        'journal': 'Proceedings of the National Academy of Sciences of the United States of America 2019',
        'authors': 'Andriotis VME, Smith AM.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047471',
          'text': '“...AT5G02490 AT5G52440 AT5G60160 AT5G24650 AT1G32900 AT5G24400 AT5G47860 AT1G64550 AT2G22170 AT2G25840 AT2G31810 AT2G35860 AT2G45060 AT2G47450 AT3G02770 AT3G15840...”',
          'title': 'Protein abundance changes and ubiquitylation targets identified after inhibition of the proteasome with syringolin A',
          'journal': 'Molecular & cellular proteomics : MCP 2014',
          'authors': 'Svozil J, Hirsch-Hoffmann M, Dudler R, Gruissem W, Baerenfaller K.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3999085',
          'text': '“...Chloroplasts and cytoplasm 6-Phosphogluconolactonase (PGL6) At5g23050 At5g24400 Arabidopsis thaliana Acyl-activating enzyme 5 (AAE5) His triad family protein...”',
          'title': 'Widespread dual targeting of proteins in land plants: when, where, how and why',
          'journal': 'Plant signaling & behavior 2013',
          'authors': 'Carrie C, Whelan J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355810',
          'text': '“...intermembrane space assembly machinery 40 SKL LOC_Os04g44550 PKL Carrie et al. ( 2010 ) 6PGL At5g24400 6-Phosphoglu conolactonase SKL LOC_Os08g43370 SSI Reumann et al. ( 2007 ) LACS7 At5g27600 Long-chain acyl-CoA synthetase 7 RLx 5 HI/SKL LOC_Os11g04980 RLx 5 HL/SKL Eubel et al. ( 2008 ),...”',
          'title': 'Defining the plant peroxisomal proteome: from Arabidopsis to rice',
          'journal': 'Frontiers in plant science 2011',
          'authors': 'Kaur N, Hu J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2944999',
          'text': '“...At4g23100 C gamma-glutamylcysteine synthetase # (M) [ 88 ] [ 89 ] [ 90 ] At5g24400 C 6-phosphogluconolactonase (M) [ 91 ] [ 91 ] At5g52920 C4 similar to pyruvate kinase isozyme G # (M) [ 92 ] [ 92 ] At2g19450 C4 diacylglycerol O-acyltransferase /...”',
          'title': 'Indispensable Roles of Plastids in Arabidopsis thaliana Embryogenesis',
          'journal': 'Current genomics 2010',
          'authors': 'Hsu SC, Belmonte MF, Harada JJ, Inoue K.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2935873',
          'text': '“...8.1 14.6 SrDW SrDW1.1 At1g43780 55.6 5.1 14.5 SrDW1.2 At1g71350 102.7 4.3 12 + SrDW5.1 At5g24400 26.8 4.4 12.3 Positive (+) effects indicate that Sq-1 alleles at that marker increased levels of caesium and strontium ions, negative () effects indicate that Sorbo alleles increased the caesium...”',
          'title': 'Caesium and strontium accumulation in shoots of Arabidopsis thaliana: genetic and physiological aspects',
          'journal': 'Journal of experimental botany 2010',
          'authors': 'Kanter U, Hauser A, Michalke B, Dräxl S, Schäffner AR.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243405',
          'text': '“...dehydrogenase 17 of 41 (At3g02360), and 6-phosphoglucono-lactonase (At5g24400) (Nyathi and Baker, 2006; Reumann et al., 2007; Eubel et al., 2008; Reumann...”',
          'title': 'Peroxisome biogenesis and function',
          'journal': 'The arabidopsis book 2009',
          'authors': 'Kaur N, Reumann S, Hu J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6804568',
          'text': '“...subunit beta-2 Q9C6Z3 2.246 1.168 ALDH2B4 Aldehyde dehydrogenase family 2 member B4 Q9SU63 3.503 1.809 EMB2024 Probable 6-phosphogluconolactonase 5 Q84WW2 6.350 2.667 LPAT1 1-acyl-sn-glycerol-3-phosphate acyltransferase 1 Q8GXU8 0.499 1.003 AOC2 Allene oxide cyclase 2 Q9LS02 0.318 1.653 Metabolites Metabolite Adducts VIP Fold-change P-value Sucrose M+Na 5.28798...”',
          'title': 'Integrated physiologic, proteomic, and metabolomic analyses of Malus halliana adaptation to saline-alkali stress',
          'journal': 'Horticulture research 2019',
          'authors': 'Jia XM, Zhu YF, Hu Y, Zhang R, Cheng L, Zhu ZL, Zhao T, Zhang X, Wang YX.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6660741',
          'text': '“...Ler +1017 At5g24400 PGL3 6-Phosphogluconolactonase 6 pgl3-1 SALK_005685 Col +975 pgl3-2 FLAG_219G10 WS +250 pgl3-3 emb2024 Col +215/437 At1g64190 PGD1 6-Phosphogluconate dehydrogenase 7 pgd1-1 GABI_762C02 Col +11 pgd1-2 SALK_121521 Col +1445 NA SALK_002712 Col 107 At5g41670 PGD3 6-Phosphogluconate dehydrogenase 7 pgd3-1 SAIL_528_E08 Col +20 pgd3-2 SALK_040050...”',
          'title': 'The plastidial pentose phosphate pathway is essential for postglobular embryo development in Arabidopsis',
          'journal': 'Proceedings of the National Academy of Sciences of the United States of America 2019',
          'authors': 'Andriotis VME, Smith AM.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6369206',
          'text': '“...proteins assisting pentose-phosphate metabolism displayed a mixed quantitative trend overtime, i.e., 6-phosphogluconate dehydrogenase (AT3G02360), 6-phosphogluconolactonase (EMB2024), and transketolase isoforms (AT3G60750 and AT2G45290). In the whole, the quantitative behavior of specific proteins involved in glycolysis, the Krebs cycle, alcoholic fermentation, energy production and additional carbon metabolism pathways...”',
          'title': 'Unveiling Kiwifruit Metabolite and Protein Changes in the Course of Postharvest Cold Storage',
          'journal': 'Frontiers in plant science 2019',
          'authors': 'Salzano AM, Renzone G, Sobolev AP, Carbone V, Petriccione M, Capitani D, Vitale M, Novi G, Zambrano N, Pasquariello MS, Mannina L, Scaloni A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4549554',
          'text': '“...( BSMDR ) and a 6-phosphoglucunolactonase encoded by a gene product embryo defective 2024 ( EMB2024 ) also showed little change in expression during culture. Values are means SEM with n 3. ( p < 0.05, p < 0.01, t -tests). Ascorbate and Glutathione are Abundant...”',
          'title': 'Peroxisomes contribute to reactive oxygen species homeostasis and cell division induction in Arabidopsis protoplasts',
          'journal': 'Frontiers in plant science 2015',
          'authors': 'Tiew TW, Sheahan MB, Rose RJ.'}],
      'url': 'http://www.uniprot.org/uniprot/Q84WW2',
      'db': 'SwissProt',
      'gene_id': 'Q84WW2',
      'identity': 0.32,
      'coverage': 0.66},
    'XP_003212309': {'organism': 'Meleagris gallopavo',
      'papers': [],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_003212309.2',
      'db': 'RefSeq',
      'gene_id': 'XP_003212309.2',
      'identity': 0.33,
      'coverage': 0.27},
    'G6PE_HUMAN / O95479': {'organism': 'Homo sapiens (Human)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5901385',
        'text': 'GeneRIF: Knockdown of H6PD resulted in an increase in ER lumen oxidation, and down-regulation of many components of the unfolded protein response, including the transcription factors activating transcription factor-4, activating transcription factor-6, split X-box binding protein-1, and CCAAT/enhancer binding protein homologous protein.',
        'title': 'Hexose-6-phosphate dehydrogenase controls cancer cell proliferation and migration through pleiotropic effects on the unfolded-protein response, calcium homeostasis, and redox balance.',
        'journal': 'FASEB journal : official publication of the Federation of American Societies for Experimental Biology 2018',
        'authors': 'Tsachaki M, Mladenovic N, Štambergová H, Birk J, Odermatt A'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4745784',
          'text': 'GeneRIF: Inhibition of glucose-6-phosphate dehydrogenase sensitizes cisplatin-resistant cells to death.',
          'title': 'Inhibition of glucose-6-phosphate dehydrogenase sensitizes cisplatin-resistant cells to death.',
          'journal': 'Oncotarget 2015',
          'authors': 'Catanzaro D, Gaude E, Orso G, Giordano C, Guzzo G, Rasola A, Ragazzi E, Caparrotta L, Frezza C, Montopoli M'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4599835',
          'text': 'GeneRIF: The G allele of rs6688832 in H6PD might exert potential genetic protective role against the development of PCOS, especially in overweight women. PCOS patients with AG genotype of rs6688832 might confer risk to the phenotype of hyperandrogenemia of PCOS.',
          'title': 'Association Analysis between the Polymorphisms of HSD11B1 and H6PD and Risk of Polycystic Ovary Syndrome in Chinese Population.',
          'journal': 'PloS one 2015',
          'authors': 'Ju R, Wu W, Tang Q, Wu D, Xia Y, Wu J, Wang X'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4021576',
          'text': 'GeneRIF: Cytoplasmic Mg2+ regulates glucose 6-phosphate utilization by reticular H6PD.',
          'title': 'Reduced cellular Mg²⁺ content enhances hexose 6-phosphate dehydrogenase activity and expression in HepG2 and HL-60 cells.',
          'journal': 'Archives of biochemistry and biophysics 2014',
          'authors': 'Voma C, Barfell A, Croniger C, Romani A'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5295240',
          'text': '“...galactosidase PGK1 P00558 Phosphoglycerate kinase 1 84.41 31 0.686 0.647 0.461 0.437 carbohydrate kinase H6PD O95479 GDH/6PGL endoplasmic bifunctional protein 35.65 23 0.708 0.729 0.477 0.483 dehydrogenase CSRP1 P21291 Cysteine and glycine-rich protein 1 64.25 9 0.711 0.670 0.426 0.427 actin family cytoskeletal protein CPVL Q9H3G5...”',
          'title': 'Quantitative Proteomic Profiling of Tachyplesin I Targets in U251 Gliomaspheres',
          'journal': 'Marine drugs 2017',
          'authors': 'Li X, Dai J, Tang Y, Li L, Jin G.'}],
      'url': 'http://www.uniprot.org/uniprot/O95479',
      'db': 'SwissProt',
      'gene_id': 'O95479',
      'identity': 0.3,
      'coverage': 0.27},
    'APA386B_1156': {'organism': 'Acetobacter pasteurianus 386B',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751514',
        'text': '“...(APA386B_1157); (20) transketolase (APA386B_1152 and APA386B_1521); (21) transaldolase (APA386B_1153) (22) glucose-6-phosphate dehydrogenase (APA386B_729); (23) 6-phosphogluconolactonase (APA386B_1156); (24) glycerol kinase (Apa396B_92); (25) glycerol-3-phosphate dehydrogenase (APA386B_1931 and APA386B_94); (26) pyruvaldehyde oxidoreductase (APA386B_1946); (27) lactoylgluthathione lyase (APA386B_2164 and APA386B_1276); (28) hydroxyacylglutathione hydrolase (APA386B_2280); (29) lactate dehydrogenase (APA386B_910 and APA386B_1053);...”',
        'title': 'Complete genome sequence and comparative analysis of Acetobacter pasteurianus 386B, a strain well-adapted to the cocoa bean fermentation ecosystem',
        'journal': 'BMC genomics 2013',
        'authors': 'Illeghems K, De Vuyst L, Weckx S.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_003623561.1',
      'db': 'RefSeq',
      'gene_id': 'WP_003623561.1',
      'identity': 0.28,
      'coverage': 0.97},
    'Gmet_2618': {'organism': 'Geobacter metallireducens GS-15',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2700814',
        'text': '“...the G. metallireducens genome includes a cluster of oxidative pentose phosphate pathway enzyme genes: 6-phosphogluconolactonase (Gmet_2618, 30% identical to the Pseudomonas putida enzyme [ 82 ]), glucose-6-phosphate dehydrogenase (Gmet_2619, 50% identical to the Nostoc punctiforme enzyme [ 83 ]), and 6-phosphogluconate dehydrogenase (Gmet_2620, 36% identical to...”',
        'title': 'The genome sequence of Geobacter metallireducens: features of metabolism, physiology and regulation common and dissimilar to Geobacter sulfurreducens',
        'journal': 'BMC microbiology 2009',
        'authors': 'Aklujkar M, Krushkal J, DiBartolo G, Lapidus A, Land ML, Lovley DR.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=1091130',
      'db': 'MicrobesOnline',
      'gene_id': '1091130',
      'identity': 0.29,
      'coverage': 0.84},
    'cg1780': {'organism': 'Corynebacterium glutamicum ATCC 13032',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3489674',
        'text': '“...cg1411 rbsA ABC-type ribose transporter, ATPase subunit (TC 3.A.1.2.1) 1.92 cg2636 catA Catechol 1,2-dioxygenase 1.88 cg1780 pgl 6-Phosphogluconolactonase 1.87 cg1413 rbsB ABC-type ribose transporter, substrate-binding lipoprotein (TC 3.A.1.2.1) 1.85 cg0498 hemC Porphobilinogen deaminase 1.84 cg2665 Hypothetical protein 1.82 cg2181* ABC-type putative dipeptide/oligopeptide transporter, substrate-binding lipoprotein 1.80...”',
        'title': 'Transcriptional regulation of the operon encoding stress-responsive ECF sigma factor SigH and its anti-sigma factor RshA, and control of its regulatory network in Corynebacterium glutamicum',
        'journal': 'BMC genomics 2012',
        'authors': 'Busche T, Silar R, Pičmanová M, Pátek M, Kalinowski J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2230225',
          'text': '“...0.88 0.89 1.09 1.15 0.74 cg1776 Transaldolase, tal 2.01 1.21 0.81 1.06 1.02 1.12 0.97 cg1780 6-phosphogluconolactonase, devB 2.51 1.28 0.89 1.14 1.02 0.91 1.08 cg1774 Transketolase, tkt 2.87 1.13 0.83 1.07 1.09 1.03 1.12 cg2836 Succinyl-CoA synthetase chain, sucD 4.48 0.93 0.52 1.41 0.75 0.81...”',
          'title': 'Co-ordinated regulation of gluconate catabolism and glucose uptake in Corynebacterium glutamicum by two functionally equivalent transcriptional regulators, GntR1 and GntR2',
          'journal': 'Molecular microbiology 2008',
          'authors': 'Frunzke J, Engels V, Hasenbein S, Gätgens C, Bott M.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767432',
          'text': '“...phosphoglyceromutase (NCgl0390); Eno, enolase (NCgl0935); Pyk, pyruvate kinase (NCgl2008); Zwf, glucose6phosphate 1dehydrogenase (NCgl1514); DevB, 6phosphogluconolactonase (NCgl1516); Gnd, 6phosphogluconate dehydrogenase (NCgl1396); Rpi, ribose5phosphate isomerase B (NCgl2337); Tkt, transketolase (NCgl1512); Tal, transaldolase (NCgl1513); AceE, pyruvate dehydrogenase subunit E1 (NCgl2167); GltA, type II citrate synthase (NCgl0795); Acn, aconitate hydratase...”',
          'title': 'Altered acetylation and succinylation profiles in Corynebacterium glutamicum in response to conditions inducing glutamate overproduction',
          'journal': 'MicrobiologyOpen 2016',
          'authors': 'Mizuno Y, Nagano-Shoji M, Kubo S, Kawamura Y, Yoshida A, Kawasaki H, Nishiyama M, Yoshida M, Kosono S.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=5916711',
      'db': 'MicrobesOnline',
      'gene_id': '5916711',
      'identity': 0.32,
      'coverage': 0.82},
    '6PGL1_ARATH / Q9LMX8': {'organism': 'Arabidopsis thaliana (Mouse-ear cress)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6375920',
        'text': '“...EC 1.1.1.41 0.57 4.29E02 gi|641851263 AT1G68750 PPC4 Phosphoenolpyruvate carboxylase 4 EC 4.1.1.31 0.63 3.05E02 gi|641851457 AT1G13700 PGL1 6-Phosphogluconolactonase 1 EC 3.1.1.31 0.54 3.89E03 gi|641853224 AT1G48030 mtLPD1 Mitochondrial lipoamide dehydrogenase 1 EC 1.8.1.4 0.65 5.78E05 gi|641853915 AT5G15490 UGD3 UDP-glucose dehydrogenase EC 1.1.1.22 0.58 2.33E02 gi|641854939 AT3G14310 PME3...”',
        'title': "Proteomic and metabolomic analyses provide insight into the off-flavour of fruits from citrus trees infected with 'Candidatus Liberibacter asiaticus'",
        'journal': 'Horticulture research 2019',
        'authors': 'Yao L, Yu Q, Huang M, Hung W, Grosser J, Chen S, Wang Y, Gmitter FG.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5710523',
          'text': '“...glucosamine/galactosamine-6-phosphate isomerase family protein gene AT1G13700 and the AT3G45730 AT3G04640 AT1G05575 AT5G49450 AT1G02660 AT1G19530 AT3G10930...”',
          'title': 'Cell growth and homeostasis are disrupted in arabidopsis rns2-2 mutants missing the main vacuolar RNase activity',
          'journal': 'Annals of botany 2017',
          'authors': 'Morriss SC, Liu X, Floyd BE, Bassham DC, MacIntosh GC.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5440562',
          'text': '“...TPS11 II-III-IV At1g54710 ATG18H II At4g24040 TREHALASE III At3g62770 ATG18A III At5g54730 ATG18F II-III Metabolism At1g13700 6-PHOSPHOGLUCONOLACTONASE 1 IV Lipid degradation At1g54100 ALDEHYDE DEHYDROGENASE H7B4 II-III At4g16760 ACYL-COENZYME A OXIDASE 1 (ACX1) III At3g23920 -AMYLASE 1 (BAM 1) II At5g65110 ACX2 III At5g55700 BAM 4 II-III...”',
          'title': 'A Conserved Carbon Starvation Response Underlies Bud Dormancy in Woody and Herbaceous Species',
          'journal': 'Frontiers in plant science 2017',
          'authors': 'Tarancón C, González-Grandío E, Oliveros JC, Nicolas M, Cubas P.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3600680',
          'text': '“...Solyc03g034020 A_96_P202249 BI925250 AT4G36550 binding / ubiquitin-protein ligase 2.1 0.014 1.6 0.009 Solyc01g010250 A_96_P215689 BP892102 AT1G13700 glucosamine/galactosamine-6-phosphate isomerase family protein 2.1 0.006 1.7 0.002 Solyc12g042470 A_96_P109252 TA51045_4081 AT1G59950 aldo/keto reductase, putative 4.2 0.018 3.8 0.045 Solyc00g071180 A_96_P016051 AF083253 AT3G12490 ATCYSB | cysteine protease inhibitor, putative /...”',
          'title': 'Expression profiling of tomato pre-abscission pedicels provides insights into abscission zone properties including competence to respond to abscission signals',
          'journal': 'BMC plant biology 2013',
          'authors': 'Nakano T, Fujisawa M, Shima Y, Ito Y.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697946',
          'text': '“...protein At5g58840 1.8 Subtilisin-like serine endopeptidase family protein At4g12910 1.4 scpl20, serine carboxy peptidase-like 20 At1g13700 Carbohydrate metabolism 1.5 Glucosamine/galacto samine-6-phosphate isomerase family protein At1g19610 Stress+redox 2.0 LCR78, PDF1.4, pathogenesis-related protein/plant defensin (PDF) family At1g22900 1.7 Disease resistance-responsive (dirigent-like protein)/molecular function unknown At2g15120 4.9 Pseudogene, disease-...”',
          'title': 'Iron-dependent modifications of the flower transcriptome, proteome, metabolome, and hormonal content in an Arabidopsis ferritin mutant',
          'journal': 'Journal of experimental botany 2013',
          'authors': 'Sudre D, Gutierrez-Carbonell E, Lattanzio G, Rellán-Álvarez R, Gaymard F, Wohlgemuth G, Fiehn O, Alvarez-Fernández A, Zamarreño AM, Bacaicoa E, Duy D, García-Mina JM, Abadía J, Philippar K, López-Millán AF, Briat JF.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355747',
          'text': '“...DUF1005 (InterPro:IPR010410) 263236_at 0.53 AT1G10470 ARR4 (RESPONSE REGULATOR 4); transcription regulator/two-component response regulator 256098_at 0.50 AT1G13700 Glucosamine/galactosamine-6-phosphate isomerase family protein 259466_at 0.44 AT1G19050 ARR7 (RESPONSE REGULATOR 7); transcription regulator/two-component response regulator 260662_at 0.49 AT1G19540 Isoflavone reductase. putative 260856_at 0.46 AT1G21910 AP2 domain-containing transcription factor family protein...”',
          'title': 'The bZIP Transcription Factor PERIANTHIA: A Multifunctional Hub for Meristem Control',
          'journal': 'Frontiers in plant science 2011',
          'authors': 'Maier AT, Stehling-Sun S, Offenburger SL, Lohmann JU.'}],
      'url': 'http://www.uniprot.org/uniprot/Q9LMX8',
      'db': 'SwissProt',
      'gene_id': 'Q9LMX8',
      'identity': 0.31,
      'coverage': 0.86},
    'SYNPCC7002_A0928': {'organism': 'Synechococcus sp. PCC 7002',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3133671',
        'text': '“...1.62 0.50 0.83 pgi Glucose-6-phosphate isomerase SYNPCC7002_A1459 0.25 3.08 2.64 0.85 0.86 zwf Glucose-6-phosphate 1-dehydrogenase SYNPCC7002_A0928 0.33 2.54 0.32 0.13 0.79 pgl 6-Phosphogluconolactonase SYNPCC7002_A0221 0.39 3.55 3.46 0.97 0.65 gnd 6-Phosphogluconate DH, decarboxylating SYNPCC7002_A1269 $ 2.45 0.09 0.06 0.66 1.39 rpiA Ribose 5-phosphate isomerase A SYNPCC7002_A0324...”',
        'title': 'Transcription Profiling of the Model Cyanobacterium Synechococcus sp. Strain PCC 7002 by Next-Gen (SOLiD™) Sequencing of cDNA',
        'journal': 'Frontiers in microbiology 2011',
        'authors': 'Ludwig M, Bryant DA.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=5418447',
      'db': 'MicrobesOnline',
      'gene_id': '5418447',
      'identity': 0.28,
      'coverage': 0.96},
    'GOX1707': {'organism': 'Gluconobacter oxydans 621H',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6191907',
        'text': '“...complex (GOX1073); pdc , pyruvate decarboxylase (GOX1081); pgk , phosphoglycerate kinase (GOX0507); pgl , 6-phosphogluconolactonase (GOX1707); ppc , phosphoenolpyruvate carboxylase (GOX0102); pyk , pyruvate kinase (GOX2250); rpe , ribulosephosphate 3-epimerase (GOX1352); rpi , ribose 5-phosphate isomerase (GOX1708); sldA , polyol dehydrogenase subunit SldA (GOX0854); sldB ,...”',
        'title': 'Global mRNA decay and 23S rRNA fragmentation in Gluconobacter oxydans 621H',
        'journal': 'BMC genomics 2018',
        'authors': 'Kranz A, Steinmann A, Degner U, Mengus-Kaya A, Matamouros S, Bott M, Polen T.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=857324',
      'db': 'MicrobesOnline',
      'gene_id': '857324',
      'identity': 0.31,
      'coverage': 0.78},
    'sll1479': {'organism': 'Synechocystis sp. PCC 6803',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959675',
        'text': '“...cycle under autotrophic conditions. Genes such as tktA ( sll1070 ), encoding transketolase, devB ( sll1479 ), encoding 6-phosphogluconolactonase, and cfxE ( sll0807 ), encoding pentose-5-phosphate-3-epimerase, were upregulated in the earlier light period (with peaks at L1), while rpiA ( slr0194 ), encoding ribose 5-phosphate isomerase,...”',
        'title': 'Diurnal Regulation of Cellular Processes in the Cyanobacterium Synechocystis sp. Strain PCC 6803: Insights from Transcriptomic, Fluxomic, and Physiological Analyses',
        'journal': 'mBio 2016',
        'authors': "Saha R, Liu D, Hoynes-O'Connor A, Liberton M, Yu J, Bhattacharyya-Pakrasi M, Balassy A, Zhang F, Moon TS, Maranas CD, Pakrasi HB."},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3196541',
          'text': '“...sll1196 * Phosphofructokinase Glycolysis 32 F sll1234 # Adenosylhomocysteinase Amino acids and amines 33 F sll1479 * 6-phosphogluconolactonase Pentose phosphate pathway 34 F slr0301 Phosphoenolpyruvate synthase Pyruvate metabolism 35 F slr0394 # Phosphoglycerate kinase Glycolysis 36 F slr0884 * , # GAPDH 1 Glycolysis 37 F...”',
          'title': 'Time-course analysis of cyanobacterium transcriptome: detecting oscillatory genes',
          'journal': 'PloS one 2011',
          'authors': 'Layana C, Diambra L.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1064041',
          'text': '“...circadian expression patterns: 6-phosphogluconolactonase (Glc, sll1479), transaldolase (TalB, slr1793), and the rate-limiting NADPH-producing enzymes...”',
          'title': 'Global analysis of circadian expression in the cyanobacterium Synechocystis sp. strain PCC 6803',
          'journal': 'Journal of bacteriology 2005',
          'authors': 'Kucho K, Okamoto K, Tsuchiya Y, Nomura S, Nango M, Kanehisa M, Ishiura M.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=13305',
      'db': 'MicrobesOnline',
      'gene_id': '13305',
      'identity': 0.31,
      'coverage': 0.83},
    '6PGL5_ARATH / Q8LEV7': {'organism': 'Arabidopsis thaliana (Mouse-ear cress)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5512810',
        'text': '“...1.4 7.0 59.7 3.9 TRAF-like family protein At4g17470 5.3 11.3 111.4 1.9 Alpha/beta-hydrolases superfamily protein At5g24420 PGL5 3.0 8.0 48.5 2.9 Cytosolic 6-phosphogluconolactonase At3g28270 AFL1 1.0 2.0 32.0 5.6 Peripheral membrane protein, growth regulation in drought At3g44970 1.0 0.9 1.9 1.9 Cytochrome P450 superfamily protein At1g18710...”',
        'title': 'Transcriptome analysis uncovers Arabidopsis F-BOX STRESS INDUCED 1 as a regulator of jasmonic acid and abscisic acid stress gene expression',
        'journal': 'BMC genomics 2017',
        'authors': 'Gonzalez LE, Keller K, Chan KX, Gessel MM, Thines BC.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4174771',
          'text': '“...3-hydroxylase-like protein 1.28 250558_at At5g07990 xyloglucan endo-1,4-beta-D-glucanase-like protein 1.31 253628_at At4g30280 6-phosphogluconolactonase-like protein 1.34 249732_at At5g24420 reticuline oxidase-like protein 1.52 254432_at At4g20830 11-beta-hydroxysteroid dehydrogenase-like 1.55 248520_at At5g50600 UDP-glucose glucosyltransferase 1.89 260799_at At1g78270 tyrosine transaminase like protein 2.12 254232_at At4g23600 oleosin 2.22 249353_at At5g40420 leucoanthocyanidin dioxygenase-like protein...”',
          'title': 'Transcriptome Analysis of Induced Systemic Drought Tolerance Elicited by Pseudomonas chlororaphis O6 in Arabidopsis thaliana',
          'journal': 'The plant pathology journal 2013',
          'authors': 'Cho SM, Kang BR, Kim YC.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3563046',
          'text': '“...transmembrane protein At3g28300 3.70* 1.06 1.09 AT14A, transmembrane protein At3g28290 3.58* 1.12 0.60 PGL5, 6-phosphogluconolactonase At5g24420 3.56*** 0.83 0.79* List of the 50 most highly induced genes in response to S. littoralis in Col-0. Three-week-old plants were challenged for 8days with first-instar larvae. Values are calculated...”',
          'title': 'Differential Contribution of Transcription Factors to Arabidopsis thaliana Defense Against Spodoptera littoralis',
          'journal': 'Frontiers in plant science 2013',
          'authors': 'Schweizer F, Bodenhausen N, Lassueur S, Masclaux FG, Reymond P.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3142401',
          'text': '“...-0.11 0.51 AT1G71170 -0.39 0.37 AT3G49360 0.39 -0.51 AT5G24420 0.31 -0.33 AT5G24410 -0.33 0.40 AT3G04790 0.84 -0.70 AT5G44520 0.70 -0.77 AT2G01290 0.68 -0.65...”',
          'title': 'Divergent evolutionary fates of major photosynthetic gene networks following gene and whole genome duplications',
          'journal': 'Plant signaling & behavior 2011',
          'authors': 'Coate JE, Doyle JJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2845782',
          'text': '“...0.6 0.0 1.8 1.0 At4g15100 SCPL30 Serine carboxypeptidase S10 family 5.0 0.5 0.1 1.9 1.1 At5g24420 6PGL 6-phosphogluconolactonase-like 8.1 1.7 0.0 5.2 1.3 At1g76790 O-methyltransferase, family 2 9.2 0.5 0.3 1.7 0.9 At4g08870 ARGAH2 Putative arginase 9.2 1.2 0.0 3.6 1.2 At1g04710 Putative acetyl-CoA acyltransferase 9.4...”',
          'title': 'Coronatine-insensitive 1 (COI1) mediates transcriptional responses of Arabidopsis thaliana to external potassium supply',
          'journal': 'Molecular plant 2010',
          'authors': 'Armengaud P, Breitling R, Amtmann A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662457',
          'text': '“...AT5G59000 zinc finger (C3HC4-type RING finger) family protein AT4G39675 unknown protein AT5G03700 PAN domain-containing protein AT5G24420 glucosamine/galactosamine-6-phosphate isomerase-related AT2G21045 similar to unknown protein AT3G24240 leucine-rich repeat transmembrane protein kinase, putative AT5G50760 auxin-responsive family protein AT1G72260 toxin receptor binding AT3G15950 (TSA1-LIKE); unknown protein AT3G15270 SPL5 AT1G17860 trypsin...”',
          'title': 'Function annotation of an SBP-box gene in Arabidopsis based on analysis of co-expression networks and promoters',
          'journal': 'International journal of molecular sciences 2009',
          'authors': 'Wang Y, Hu Z, Yang Y, Chen X, Chen G.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2375892',
          'text': '“...At1g05550; GUS is expressed in both integuments and the nucellus, at the chalazal region. (B) At5g24420; GUS is expressed in the funiculus and both integuments throughout the ovule. (C) At5g49180; GUS is expressed in both integuments and the funiculus, but not in the female gametophyte. (D)...”',
          'title': 'A spatial dissection of the Arabidopsis floral transcriptome by MPSS',
          'journal': 'BMC plant biology 2008',
          'authors': 'Peiffer JA, Kaushik S, Sakai H, Arteaga-Vazquez M, Sanchez-Leon N, Ghazal H, Vielle-Calzada JP, Meyers BC.'}],
      'url': 'http://www.uniprot.org/uniprot/Q8LEV7',
      'db': 'SwissProt',
      'gene_id': 'Q8LEV7',
      'identity': 0.3,
      'coverage': 0.91},
    'alr1602': {'organism': 'Nostoc sp. PCC 7120',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3232833',
        'text': '“...cyanobacteria. In Anabaena sp. strain PCC 7120, both genes, alr1602 and alr1603 (fraH), can be expressed as independent transcripts as well as in a common...”',
        'title': 'FraH is required for reorganization of intracellular membranes during heterocyst differentiation in Anabaena sp. strain PCC 7120',
        'journal': 'Journal of bacteriology 2011',
        'authors': 'Merino-Puerto V, Mariscal V, Schwarz H, Maldener I, Mullineaux CW, Herrero A, Flores E.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=163955',
      'db': 'MicrobesOnline',
      'gene_id': '163955',
      'identity': 0.3,
      'coverage': 0.95},
    'cce_4743': {'organism': 'Cyanothece sp. ATCC 51142',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6400067',
        'text': '“...Oxidative PP enzymes including glucose 6-phosphate dehydrogenase; Zwf (cce_2536), 6-phosphogluconate dehydrogenase; Gnd (cce_3746), 6-phosphogluconolactonase; Pgl (cce_4743), ribulose-phosphate 3-epimerase; Rpe (cce_0798), and transaldolases; TalC (cce_4208) and TalA (cce_4687) were identified as complexes. For example, Zwf, a branching enzyme of the PP pathway was identified with R app...”',
        'title': 'Analysis of Protein Complexes in the Unicellular Cyanobacterium Cyanothece ATCC 51142',
        'journal': 'Journal of proteome research 2018',
        'authors': 'Aryal UK, Ding Z, Hedrick V, Sobreira TJP, Kihara D, Sherman LA.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3329150',
          'text': '“...ammonium- and light-limited conditions. Enzymes of the oxidative PPP branch, namely glucose-6-phosphate dehydrogenase (cce_25352536), 6-phosphogluconolactonase (cce_4743) and 6-phosphogluconate dehydrogenase (cce_3746), showed increased abundances under ammonium limited conditions. Similarly, two-fold increase in abundance levels was observed for gluconeogenesis proteins, including fructose 1,6-bisphosphatase (cce_4758), glucose-6-phosphate isomerase (cce_0666), glyceraldehyde-3-phosphate...”',
          'title': 'Genome-scale modeling of light-driven reductant partitioning and carbon fluxes in diazotrophic unicellular cyanobacterium Cyanothece sp. ATCC 51142',
          'journal': 'PLoS computational biology 2012',
          'authors': 'Vu TT, Stolyar SM, Pinchuk GE, Hill EA, Kucek LA, Brown RN, Lipton MS, Osterman A, Fredrickson JK, Konopka AE, Beliaev AS, Reed JL.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=5431433',
      'db': 'MicrobesOnline',
      'gene_id': '5431433',
      'identity': 0.29,
      'coverage': 0.98},
    'VCA0897': {'organism': 'Vibrio cholerae O1 biovar eltor str. N16961',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606805',
        'text': '“...stress No reads (0) 1.2 VC2294 nqrB Iron and oxidative stress No reads (0) 5.1 VCA0897 pgl Pentose phosphate pathway 26.3 (0.009) 2.1 VCA0609 Unknown 23.4 (1.60E15) 2.9 VCA0634 Putative tRNA modification 9.3 (0.011) 1.5 VC2517 Putative ABC-type transport 6.7 (1.00E08) 1.5 VC2234* rnhA RNase H1,...”',
        'title': 'RadD Contributes to R-Loop Avoidance in Sub-MIC Tobramycin',
        'journal': 'mBio 2019',
        'authors': 'Negro V, Krin E, Aguilar Pierlé S, Chaze T, Giai Gianetto Q, Kennedy SP, Matondo M, Mazel D, Baharoglu Z.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5816637',
          'text': '“...aminotransferase 1.820 VCA0825 ectA L-24-diaminobutyric acid acetyltransferase 1.691 VCA0867 ompW outer membrane protein OmpW 1.639 VCA0897 devB devB protein 1.127 VCA0898 gnd 6-phosphogluconate dehydrogenase decarboxylating 1.401 1.262 VCA0907 hutZ heme binding 1.430 1.047 VCA0908 hutX Unknown, linked to hutZ 1.626 1.091 VCA0909 hutW unknown, linked to...”',
          'title': 'Transcriptomics reveals a cross-modulatory effect between riboflavin and iron and outlines responses to riboflavin biosynthesis and uptake in Vibrio cholerae',
          'journal': 'Scientific reports 2018',
          'authors': 'Sepúlveda-Cisternas I, Lozano Aguirre L, Fuentes Flores A, Vásquez Solis de Ovando I, García-Angulo VA.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1201252',
          'text': '“...5.0 104 1.1 105 Energy metabolism VC2413 VC2447 VCA0897 Pyruvate dehydrogenase, E2 component, dihydrolipoamide Enolase DevB protein 3.24 3.05 2.04 3.5 104 2.8...”',
          'title': 'Transcriptional profiling of Vibrio cholerae recovered directly from patient specimens during early and late stages of human infection',
          'journal': 'Infection and immunity 2005',
          'authors': 'Larocque RC, Harris JB, Dziejman M, Li X, Khan AI, Faruque AS, Faruque SM, Nair GB, Ryan ET, Qadri F, Mekalanos JJ, Calderwood SB.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=56238',
      'db': 'MicrobesOnline',
      'gene_id': '56238',
      'identity': 0.29,
      'coverage': 0.84},
    'Rv1445c': {'organism': 'Mycobacterium tuberculosis H37Rv',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4802306',
        'text': '“...Rv1121 ) Glycolysis (PGI, TPI), Pentose Phosphate Pathway (RPE, TALA, TKT1, TKT2) PGL (6-phosphogluconolactonase) ( Rv1445c ) Glycolysis ( PGI , TPI), Pentose Phosphate Pathway (TKT1, TKT2) GLUDC (Glutamate decarboxylase) ( Rv3432c ) Alanine and Aspartate Metabolism (ASNS1), Glycolysis ( GAPD , PGK ) GND (phosphogluconate...”',
        'title': 'Integration of Metabolic Modeling with Gene Co-expression Reveals Transcriptionally Programmed Reactions Explaining Robustness in Mycobacterium tuberculosis',
        'journal': 'Scientific reports 2016',
        'authors': 'Puniya BL, Kulshreshtha D, Mittal I, Mobeen A, Ramachandran S.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=33218',
      'db': 'MicrobesOnline',
      'gene_id': '33218',
      'identity': 0.31,
      'coverage': 0.84},
    'CT154_14190': {'organism': 'Komagataeibacter xylinus',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5908849',
        'text': '“...6pgd , CT154_14710), glucose-6-phosphate 1-dehydrogenase ( 6pgd-1 , CT154_14205 and CT154_05345), 6-phosphogluconolactonase ( 6pgl , CT154_14190), 6-phosphogluconate dehydrogenase ( 6pgad , CT154_14210), ribose-5-phosphate isomerase A ( rpi A , CT154_14185), ribulose-phosphate 3-epimerase ( rpe , CT154_13290), transketolase ( tkt , CT154_07695, CT154_08455, CT154_12910, and CT154_14220), and...”',
        'title': 'Complete genome analysis of Gluconacetobacter xylinus CGMCC 2955 for elucidating bacterial cellulose biosynthesis and metabolic regulation',
        'journal': 'Scientific reports 2018',
        'authors': 'Liu M, Liu L, Jia S, Li S, Zou Y, Zhong C.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_034927080.1',
      'db': 'RefSeq',
      'gene_id': 'WP_034927080.1',
      'identity': 0.28,
      'coverage': 0.84},
    '6PGL2_ARATH / Q8LG70': {'organism': 'Arabidopsis thaliana (Mouse-ear cress)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3142401',
        'text': '“...-0.51 0.85 AT5G41670 -0.11 0.51 AT1G71170 -0.39 0.37 AT3G49360 0.39 -0.51 AT5G24420 0.31 -0.33 AT5G24410 -0.33 0.40 AT3G04790 0.84 -0.70 AT5G44520 0.70 -0.77...”',
        'title': 'Divergent evolutionary fates of major photosynthetic gene networks following gene and whole genome duplications',
        'journal': 'Plant signaling & behavior 2011',
        'authors': 'Coate JE, Doyle JJ.'}],
      'url': 'http://www.uniprot.org/uniprot/Q8LG70',
      'db': 'SwissProt',
      'gene_id': 'Q8LG70',
      'identity': 0.28,
      'coverage': 0.97},
    'AL01_06130': {'organism': 'Bombella intestini',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5112900',
        'text': '“...(AL01_07015); 34, 6-phosphogluconate dehydrogenase (AL01_06120); 35, ribose-5-phosphate isomerase (AL01_06135); 36, glucose-6-phosphate dehydrogenase (AL01_02790); 37, 6-phosphogluconolactonase (AL01_06130); 38, ribulose-phosphate 3-epimerase (AL01_09060); 39, polyol:NADP oxidoreductase (AL01_07080); 40, phosphomannomutase (AL_0102400); 41, mannose-1-phosphate guanyltransferase (AL01_07360); 42, carbohydrate kinase (AL01_03675); 43, mannose06-phosphate isomerase (AL01_00140); 44, 2-dehydro-3-deoxyphosphogluconate aldolase (AL01_04330). All genes encoding...”',
        'title': 'Whole-Genome Sequence Analysis of Bombella intestini LMG 28161T, a Novel Acetic Acid Bacterium Isolated from the Crop of a Red-Tailed Bumble Bee, Bombus lapidarius',
        'journal': 'PloS one 2016',
        'authors': 'Li L, Illeghems K, Van Kerrebroeck S, Borremans W, Cleenwerck I, Smagghe G, De Vuyst L, Vandamme P.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_077396539.1',
      'db': 'RefSeq',
      'gene_id': 'WP_077396539.1',
      'identity': 0.29,
      'coverage': 0.68},
    'SCO1939': {'organism': 'Streptomyces coelicolor A3(2)',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294797',
        'text': '“...(SCO3070; 35% aa identity to NagA), a 6-phosphogluconolactonase (SCO1939; 29% aa identity to NagB), and an FGGY family carbohydrate kinase (SCO6260; 35% aa...”',
        'title': 'Functional analysis of the N-acetylglucosamine metabolic genes of Streptomyces coelicolor and role in control of development and antibiotic production',
        'journal': 'Journal of bacteriology 2012',
        'authors': 'Świątek MA, Tenconi E, Rigali S, van Wezel GP.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3277767',
          'text': '“...pentose phosphate pathway, i.e. 6-phosphogluconolactonase (SCO1939) and a transketolase for the conversion of Derythrose-4-phosphate to...”',
          'title': 'Metabolic switches and adaptations deduced from the proteomes of Streptomyces coelicolor wild type and phoP mutant grown in batch culture',
          'journal': 'Molecular & cellular proteomics : MCP 2012',
          'authors': 'Thomas L, Hodgson DA, Wentzel A, Nieselt K, Ellingsen TE, Moore J, Morrissey ER, Legaie R, STREAM Consortium, Wohlleben W, Rodríguez-García A, Martín JF, Burroughs NJ, Wellington EM, Smith MC.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4546429',
          'text': '“...pentose phosphate pathway S1 9 SCO1937 zwf2 Glucose 6-phosphate 1-dehydrogenase 9 SCO1938 Hypothetical protein 9 SCO1939 pgl 6-phosphogluconolactonase 10 SCO1958 * uvrA ABC excision nuclease subunit A (A), S1 14 SCO1995 * Unknown (S1, A) 11 SCO1996 coaE Dephospho-CoA kinase (A) 11 SCO1997 * Unknown (S1,...”',
          'title': 'Conservation of thiol-oxidative stress responses regulated by SigR orthologues in actinomycetes',
          'journal': 'Molecular microbiology 2012',
          'authors': 'Kim MS, Dufour YS, Yoo JS, Cho YB, Park JH, Nam GB, Kim HM, Lee KL, Donohue TJ, Roe JH.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=242296',
      'db': 'MicrobesOnline',
      'gene_id': '242296',
      'identity': 0.29,
      'coverage': 0.9},
    'XP_828707': {'organism': 'Trypanosoma brucei brucei TREU927',
      'papers': [],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_828707.1',
      'db': 'RefSeq',
      'gene_id': 'XP_828707.1',
      'identity': 0.28,
      'coverage': 0.79},
    'MAP1174c': {'organism': 'Mycobacterium avium subsp. paratuberculosis str. k10',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2223459',
        'text': '“...D3 A1 H4 MAP1417c MAP2663c MAP3121 MAP4129 MAP1174c MAP4199 MAP3902c MAP2764c MAP3131 MAP0261c MAP2360c MAP3903c MAP2761c MAP1121c MAP0182c MAP2156 MAP0863...”',
        'title': 'Profiling bovine antibody responses to Mycobacterium avium subsp. paratuberculosis infection by using protein arrays',
        'journal': 'Infection and immunity 2008',
        'authors': 'Bannantine JP, Paustian ML, Waters WR, Stabel JR, Palmer MV, Li L, Kapur V.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265687',
          'text': '“...0.874 33.469 75.392 68.964 B11 MAP1121c 0.758 2.438 0 2.094 0.699 3.364 0 1.848 A10 MAP1174c 0 2.301 3.155 4.411 0.623 1.921 5.81 4.955 C11 MAP1204 0.78 40.8 43.856 55.046 0 28.367 61.122 58.985 F2 MAP1233 2.568 12.375 22.507 32.493 1.932 3.212 24.157 30.447 C5 MAP1345...”',
          'title': 'Early antibody response against Mycobacterium avium subspecies paratuberculosis antigens in subclinical cattle',
          'journal': 'Proteome science 2008',
          'authors': 'Bannantine JP, Bayles DO, Waters WR, Palmer MV, Stabel JR, Paustian ML.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=535110',
      'db': 'MicrobesOnline',
      'gene_id': '535110',
      'identity': 0.28,
      'coverage': 1.0},
    'blr6761': {'organism': 'Bradyrhizobium japonicum USDA 110',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2045231',
        'text': '“...of glucose-6phosphate dehydrogenase (blr6760), 6-phosphoglucolactonase (blr6761), and 6-phosphogluconate dehydrogenase (blr6759) (Table 2), all of which encode...”',
        'title': 'Transcriptional and physiological responses of Bradyrhizobium japonicum to desiccation-induced stress',
        'journal': 'Journal of bacteriology 2007',
        'authors': 'Cytryn EJ, Sangurdekar DP, Streeter JG, Franck WL, Chang WS, Stacey G, Emerich DW, Joshi T, Xu D, Sadowsky MJ.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=321972',
      'db': 'MicrobesOnline',
      'gene_id': '321972',
      'identity': 0.31,
      'coverage': 0.82},
    'Q9GRG6': {'organism': 'Trypanosoma brucei',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640789',
        'text': '“...AMED_2808 AORI_2795 AJAP_24815 pgi AMETH_4094 AMED_2807 AORI_2794 AJAP_24820 zwf AMETH_4095 AMED_2806 AORI_2793 AJAP_24825 pgl AMETH_4097 AMED_2804 AORI_2791 AJAP_24835 Glycolysis/gluconeogenesis glpX AMETH_5852 AMED_8059 AORI_6881 AJAP_04695 AMETH_4516 AMETH_4530 fbaA AMETH_6838 AMED_9110 AORI_7850 AJAP_40945 AMETH_4655 AMETH_6121 AMED_3923 AORI_5186 AJAP_13540 AMETH_4531 Each line is the corresponding ortholog. On the other...”',
        'title': 'A systematic study of the whole genome sequence of Amycolatopsis methanolica strain 239T provides an insight into its physiological and taxonomic properties which correlate with its position in the genus',
        'journal': 'Synthetic and systems biotechnology 2016',
        'authors': 'Tang B, Xie F, Zhao W, Wang J, Dai S, Zheng H, Ding X, Cen X, Liu H, Yu Y, Zhou H, Zhou Y, Zhang L, Goodfellow M, Zhao GP.'}],
      'url': 'http://www.brenda-enzymes.org/sequences.php?AC=Q9GRG6',
      'db': 'BRENDA',
      'gene_id': 'Q9GRG6',
      'identity': 0.31,
      'coverage': 0.66},
    'ACPL_1861': {'organism': 'Actinoplanes sp. SE50/110',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4479805',
        'text': '“...pathway and the inhibition of glycolysis. Thus, genes in the pentose phosphate pathway, such as ACPL_1861, ACPL_6461, ACPL_1328, were candidates for overexpression to improve acarbose production. In addition, the flux of the reaction catalyzed by alanine aminotransferase (EC: 2.6.1.2), which forms glutamate, increased with increasing acarbose...”',
        'title': 'Reconstruction and in silico analysis of an Actinoplanes sp. SE50/110 genome-scale metabolic model for acarbose production',
        'journal': 'Frontiers in microbiology 2015',
        'authors': 'Wang Y, Xu N, Ye C, Liu L, Shi Z, Wu J.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_014688831.1',
      'db': 'RefSeq',
      'gene_id': 'WP_014688831.1',
      'identity': 0.3,
      'coverage': 0.83},
    'TP0477': {'organism': 'Treponema pallidum subsp. pallidum str. Nichols',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3373638',
        'text': '“...204 232 Yes TPChic0425 TP0425 Hypothetical protein F GG insertion 453695:454240 181 86 No TPChic0477 TP0477 Hypothetical protein R C insertion 507750:508454 234 241 No TPChic0479 TP0479 Hypothetical protein R C insertion 510135:510692 185 224 Yes TPChic0486 TP0486 Hypothetical protein R T deletion 518618:520213 581 484...”',
        'title': 'Footprint of positive selection in Treponema pallidum subsp. pallidum genome sequences suggests adaptive microevolution of the syphilis pathogen',
        'journal': 'PLoS neglected tropical diseases 2012',
        'authors': 'Giacani L, Chattopadhyay S, Centurion-Lara A, Jeffrey BM, Le HT, Molini BJ, Lukehart SA, Sokurenko EV, Rockey DD.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=26299',
      'db': 'MicrobesOnline',
      'gene_id': '26299',
      'identity': 0.26,
      'coverage': 0.85},
    'AORI_2791': {'organism': 'Amycolatopsis keratiniphila',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640789',
        'text': '“...AORI_2795 AJAP_24815 pgi AMETH_4094 AMED_2807 AORI_2794 AJAP_24820 zwf AMETH_4095 AMED_2806 AORI_2793 AJAP_24825 pgl AMETH_4097 AMED_2804 AORI_2791 AJAP_24835 Glycolysis/gluconeogenesis glpX AMETH_5852 AMED_8059 AORI_6881 AJAP_04695 AMETH_4516 AMETH_4530 fbaA AMETH_6838 AMED_9110 AORI_7850 AJAP_40945 AMETH_4655 AMETH_6121 AMED_3923 AORI_5186 AJAP_13540 AMETH_4531 Each line is the corresponding ortholog. On the other hand,...”',
        'title': 'A systematic study of the whole genome sequence of Amycolatopsis methanolica strain 239T provides an insight into its physiological and taxonomic properties which correlate with its position in the genus',
        'journal': 'Synthetic and systems biotechnology 2016',
        'authors': 'Tang B, Xie F, Zhao W, Wang J, Dai S, Zheng H, Ding X, Cen X, Liu H, Yu Y, Zhou H, Zhou Y, Zhang L, Goodfellow M, Zhao GP.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_016333147.1',
      'db': 'RefSeq',
      'gene_id': 'WP_016333147.1',
      'identity': 0.28,
      'coverage': 0.81},
    'CC2056': {'organism': 'Caulobacter crescentus CB15',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1855722',
        'text': '“...Glycolysis and gluconeogenesis Xyl Gluc zwf (CC2057) CC2056 ppc (CC1493) CC3065 1 1 2 1 Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)...”',
        'title': 'Genetic analysis of a novel pathway for D-xylose metabolism in Caulobacter crescentus',
        'journal': 'Journal of bacteriology 2007',
        'authors': 'Stephens C, Christen B, Fuchs T, Sundaram V, Watanabe K, Jenal U.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC344409',
          'text': '“...glk CC2055, phosphogluconate dehydratase, edd CC2056, 6-phosphogluconolactonase, pgl CC2057, glucose-6-phosphate 1-dehydrogenase, zwf Transport CC0028,...”',
          'title': 'Transcriptional profiling of Caulobacter crescentus during growth on complex and minimal media',
          'journal': 'Journal of bacteriology 2004',
          'authors': 'Hottes AK, Meewan M, Yang D, Arana N, Romero P, McAdams HH, Stephens C.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1170934',
          'text': '“...Biolabs (Beverly, MA). Bacterial strains and bacteriophage Strain CC2056 [xis6ind - recA56 trpam thi lacam (Radnedge et al., 1996)] was used for the partition...”',
          'title': 'Probing the structure of complex macromolecular interactions by homolog specificity scanning: the P1 and P7 plasmid partition systems',
          'journal': 'The EMBO journal 1998',
          'authors': 'Radnedge L, Youngren B, Davis M, Austin S.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=98871',
      'db': 'MicrobesOnline',
      'gene_id': '98871',
      'identity': 0.27,
      'coverage': 0.94},
    'AJAP_24835': {'organism': 'Amycolatopsis japonica',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640789',
        'text': '“...AJAP_24815 pgi AMETH_4094 AMED_2807 AORI_2794 AJAP_24820 zwf AMETH_4095 AMED_2806 AORI_2793 AJAP_24825 pgl AMETH_4097 AMED_2804 AORI_2791 AJAP_24835 Glycolysis/gluconeogenesis glpX AMETH_5852 AMED_8059 AORI_6881 AJAP_04695 AMETH_4516 AMETH_4530 fbaA AMETH_6838 AMED_9110 AORI_7850 AJAP_40945 AMETH_4655 AMETH_6121 AMED_3923 AORI_5186 AJAP_13540 AMETH_4531 Each line is the corresponding ortholog. On the other hand, in...”',
        'title': 'A systematic study of the whole genome sequence of Amycolatopsis methanolica strain 239T provides an insight into its physiological and taxonomic properties which correlate with its position in the genus',
        'journal': 'Synthetic and systems biotechnology 2016',
        'authors': 'Tang B, Xie F, Zhao W, Wang J, Dai S, Zheng H, Ding X, Cen X, Liu H, Yu Y, Zhou H, Zhou Y, Zhang L, Goodfellow M, Zhao GP.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_038515527.1',
      'db': 'RefSeq',
      'gene_id': 'WP_038515527.1',
      'identity': 0.27,
      'coverage': 0.81},
    'BBMN68_1187': {'organism': 'Bifidobacterium longum subsp. longum BBMN68',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517610',
        'text': '“...Gene Gene description BBMN68_1185 237.79 512.44 2.16 * G COG0364 zwf K00036, glucose-6-phosphate 1-dehydrogenase [EC:1.1.1.49] BBMN68_1187 80.61 173.56 2.15 * G COG0363 magB K01057, 6-phosphogluconolactonase [EC:3.1.1.31] BBMN68_1188 815.25 1577.33 1.93 G COG0362 gnd K00033, 6-phosphogluconate dehydrogenase [EC:1.1.1.44] BBMN68_408 1291.78 1065.78 1.21 G COG0021 tkTA K00615, transketolase...”',
        'title': 'Mechanism analysis of acid tolerance response of bifidobacterium longum subsp. longum BBMN 68 by gene expression profile using RNA-sequencing',
        'journal': 'PloS one 2012',
        'authors': 'Jin J, Zhang B, Guo H, Cui J, Jiang L, Song S, Sun M, Ren F.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_077321619.1',
      'db': 'RefSeq',
      'gene_id': 'WP_077321619.1',
      'identity': 0.26,
      'coverage': 0.67},
    'BAB2_0459': {'organism': 'Brucella melitensis biovar Abortus 2308',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5587712',
        'text': '“...NO 1.86 0.16 Pentose Phosphate Shunt GI:82700516 BAB1_1740 tkt transketolase C NO 1.25 0.04 GI:83269362 BAB2_0459 pgl 6-phosphogluconolactonase U NO 0.63 0.05 GI:83269363 BAB2_0460 glucose-6-phosphate 1-dehydrogenase U NO 0.70 0.09 ATP-binding cassette ABC transporter complex GI:82699142 BAB1_0238 ABC-type sugar transport system, periplasmic comp. U YES 0.60...”',
        'title': 'Iron-dependent reconfiguration of the proteome underlies the intracellular lifestyle of Brucella abortus',
        'journal': 'Scientific reports 2017',
        'authors': 'Roset MS, Alefantis TG, DelVecchio VG, Briones G.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2877026',
          'text': '“...Unknown BAB1_0540 Formyl transferase, BAB1_1170 GltA BAB1_1840 MmsA BAB2_0457 FolD N-terminal BAB1_1174 FabZ BAB1_1872 PrfA BAB2_0459 Pgl BAB1_0544 DegT/DnrJ/EryC1/StrS BAB1_1187 Endoribonuclease BAB1_1874 LysC BAB2_0460 Zwf BAB1_0561 Man-6-P isomerase L-PSP BAB1_1879 GrxC BAB2_0483 ShuT type II BAB1_1188 GDPD BAB1_1887 HemC BAB2_0513 GcvT BAB1_0570 XylA BAB1_1205 ElaB-domain BAB1_1895...”',
          'title': 'Proteomics-based confirmation of protein expression and correction of annotation errors in the Brucella abortus genome',
          'journal': 'BMC genomics 2010',
          'authors': 'Lamontagne J, Béland M, Forest A, Côté-Martin A, Nassif N, Tomaki F, Moriyón I, Moreno E, Paramithiotis E.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=1055506',
      'db': 'MicrobesOnline',
      'gene_id': '1055506',
      'identity': 0.29,
      'coverage': 0.84},
    'RL0752': {'organism': 'Rhizobium leguminosarum bv. viciae 3841',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5165102',
        'text': '“...Berkeley Locus pRL80070 pRL120710 RL0037 RL0054 RL0182 RL0183 RL0751 RL0752 RL0753 RL0761 RL1010 RL1351 RL2234 RL2239 RL2241 RL2243 Role of O2 in the Growth of...”',
        'title': 'Role of O2 in the Growth of Rhizobium leguminosarum bv. viciae 3841 on Glucose and Succinate',
        'journal': 'Journal of bacteriology 2017',
        'authors': 'Wheatley RM, Ramachandran VK, Geddes BA, Perry BJ, Yost CK, Poole PS.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5118466',
          'text': '“...1-dehydrogenase PGN 1 19 NE 1 14.5 NE 6.3.0 Putative XRE family transcriptional regulator 1.2 RL0752 pgl 6-phosphogluconolactonase GI 0 0 ES 0 0 ES 3.5.6 Putative 6-phosphogluconolactonase 1.3 RL0751 edd Phosphogluconate dehydratase VGI 0 0 ES 1 8.73 NE 3.3.3 Putative phosphogluconate dehydratase 1.4 RL4162...”',
          'title': 'The Use of Transposon Insertion Sequencing to Interrogate the Core Functional Genome of the Legume Symbiont Rhizobium leguminosarum',
          'journal': 'Frontiers in microbiology 2016',
          'authors': 'Perry BJ, Akter MS, Yost CK.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=2203022',
      'db': 'MicrobesOnline',
      'gene_id': '2203022',
      'identity': 0.24,
      'coverage': 0.83},
    'PP1023, PP_1023': {'organism': 'Pseudomonas putida KT2440',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4855977',
        'text': '“...0.85 0.09 zwf2 PP4042 0.48 0.14 0.10 0.48 zwf3 PP5351 0.23 0.05 0.10 0.21 pgl PP1023 6-P-gluconate dehydrogenase 0.65 0.42 1.15 0.77 gnd PP4043 6-P-gluconate dehydrogenase 0.08 0.15 0.1 0.23 gnuK PP3416 Carbohydrate kinase 0.06 0.52 0.10 0.7 kguK PP3378 Dehydroglucokinase 0.02 0.63 0.10 0.21 kguD...”',
        'title': 'Integrated analysis of gene expression and metabolic fluxes in PHA-producing Pseudomonas putida grown on glycerol',
        'journal': 'Microbial cell factories 2016',
        'authors': 'Beckers V, Poblete-Castro I, Tomasch J, Wittmann C.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4077159',
          'text': '“...zwf1 PP1022 G6P dehydrogenase 5.2 3.6 zwf2 PP4042 0.2 -0.6 zwf3 PP5351 -0.1 -0.3 pgl PP1023 6-phosphogluconate dehydrogenase 9.3 7.8 gnd PP4043 6-phosphoglucolacto dehydrogenase 0.2 -0.3 gnuK PP3416 Carbohydrate kinase -0.1 0.3 kguK PP3378 Dehydroglucokinase 0.6 0.5 kguD PP3376 2-Ketogluconate 6-phosphate reductase 0.1 -0.1 rpiA PP5150...”',
          'title': 'Production of medium chain length polyhydroxyalkanoate in metabolic flux optimized Pseudomonas putida',
          'journal': 'Microbial cell factories 2014',
          'authors': 'Borrero-de Acuña JM, Bielecka A, Häussler S, Schobert M, Jahn M, Wittmann C, Jahn D, Poblete-Castro I.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2293218',
          'text': '“...organizationa PP1009 PP1010 PP1011 PP1012 PP1022 PP1023 PP1024 PP4488 gap-1 edd glk gltR-2 zwf-1 pgl eda Glyceraldehyde-3-P-dehydrogenase 6-Phosphogluconate...”',
          'title': 'A set of activators and repressors control peripheral glucose pathways in Pseudomonas putida to yield a common central intermediate',
          'journal': 'Journal of bacteriology 2008',
          'authors': 'del Castillo T, Duque E, Ramos JL.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563689',
          'text': '“...22.26 0.83 11.16 1 Triose-phosphate isomerase PP4715 d -Fructose 7.7 29.74 0.83 3.5 6-Phosphoglucono lactonase PP1023 2 Glucose dehydrogenase (membrane) PP1444 d -Glucose 7.05 28.51 0.83 4.17 6-Phosphoglucono lactonase PP1023 3 Isocitrate dehydrogenase PP4011 or PP4012 l -Serine 22.41 23.01 6.66 10.67 Formate dehydrogenase PP0490 or...”',
          'title': 'Genome-scale reconstruction and analysis of the Pseudomonas putida KT2440 metabolic network facilitates applications in biotechnology',
          'journal': 'PLoS computational biology 2008',
          'authors': 'Puchałka J, Oberhardt MA, Godinho M, Bielecka A, Regenhardt D, Timmis KN, Papin JA, Martins dos Santos VA.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2583600',
          'text': '“...metabolism PP_0794 (fruK) PP_1010 (edd) PP_1023 (pgl) 1-Phosphofructokinase 6-Phosphogluconate dehydratase 6-Phosphogluconolactonase 2.8 2.6 2.2 Amino acid...”',
          'title': 'A two-component regulatory system integrates redox state and population density sensing in Pseudomonas putida',
          'journal': 'Journal of bacteriology 2008',
          'authors': 'Fernández-Piñar R, Ramos JL, Rodríguez-Herva JJ, Espinosa-Urgel M.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797225',
          'text': '“...PP0290 PP0291 PP0379 PP0663 PP0700 PP0794 PP0861 PP1009 PP1023 PP1024 PP1399 PP1400 PP1545 PP1673 PP1827 PP1891 PP2307 PP2310 PP2794 PP2885 PP2886 PP2887 PP3397...”',
          'title': 'The RpoT regulon of Pseudomonas putida DOT-T1E and its role in stress endurance against solvents',
          'journal': 'Journal of bacteriology 2007',
          'authors': 'Duque E, Rodríguez-Herva JJ, de la Torre J, Domínguez-Cuevas P, Muñoz-Rojas J, Ramos JL.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC532441',
          'text': '“...regulator HexR, PP1022 determines glucose-6-phosphate dehydrogenase, PP1023 encodes phosphogluconolactonase, and PP1025 corresponds to leuA, the gene for...”',
          'title': 'Genetic evidence that catabolites of the Entner-Doudoroff pathway signal C source repression of the sigma54 Pu promoter of Pseudomonas putida',
          'journal': 'Journal of bacteriology 2004',
          'authors': 'Velázquez F, di Bartolo I, de Lorenzo V.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=216720',
      'db': 'MicrobesOnline',
      'gene_id': '216720',
      'identity': 0.25,
      'coverage': 0.89},
    'Q9X2N2': {'organism': 'Pseudomonas aeruginosa',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6591848',
        'text': '“...0.00038 Aspartate ammonia-lyase aspA PA5429 2.56 0.0029 Sulfite reductase cysI PA1838 2.47 0.0029 6-phosphogluconolactonase pgI PA3182 2.44 0.0018 Soluble pyridine nucleotide transhydrogenase sth PA2991 2.35 0.0041 Probable ferredoxin PA1551* 2.29 ID327 0.0017 Cytochrome C-oxidase, cbb3-type, CcoP subunit ccoP2 PA1555* 2.22 0.0041 Probable cytochrome c oxidase subunit...”',
        'title': 'Seawater salt-trapped Pseudomonas aeruginosa survives for years and gets primed for salinity tolerance',
        'journal': 'BMC microbiology 2019',
        'authors': 'Elabed H, González-Tortuero E, Ibacache-Quiroga C, Bakhrouf A, Johnston P, Gaddour K, Blázquez J, Rodríguez-Rojas A.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6814747',
          'text': '“...2.1 MaiA PA2007 Maleylacetoacetate isomerase 5 39.2% 2.0 HutI PA5092 Imidazolonepropionase 3 11.2% 1.8 Pgl PA3182 6-phosphogluconolactonase 9 52.1% 1.5 Hfq PA4944 RNA-binding protein in P. aeruginosa 16 51.3% 1.5 Biofilm LldD PA4771 L-lactate dehydrogenase 12 39.60% 5.3 FolE2 PA5539 GTP cyclohydrolase 6 36.20% 2.9 PvdA...”',
          'title': 'Reverse diauxie phenotype in Pseudomonas aeruginosa biofilm revealed by exometabolomics and label-free proteomics',
          'journal': 'NPJ biofilms and microbiomes 2019',
          'authors': 'Yung YP, McGill SL, Chen H, Park H, Carlson RP, Hanley L.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6444534',
          'text': '“...PA3920 4.0 0.003 2 Probable metal transporting P-type ATPase Cytoplasmic Membrane [Class 3] Class 3 PA3182 3.0 0.008 2 6-phosphogluconolactonase Unknown [Class 3] Class 1 PA0283 2.8 0.009 10 Sulfate-binding protein precursor Periplasmic [Class 2]; Periplasmic [Class 3] Class 2 PA3450 2.8 0.001 6 Probable antioxidant...”',
          'title': 'Proteome profiling of Pseudomonas aeruginosa PAO1 identifies novel responders to copper stress',
          'journal': 'BMC microbiology 2019',
          'authors': 'Wright BW, Kamath KS, Krisp C, Molloy MP.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6235884',
          'text': '“...FAD-dependent oxidoreductase 1,55 0,2 2,13 CCM/catabolism eda PA3181 2-keto-3-deoxy-6-phosphogluconate aldolase 0,29 1,6 4,84 CCM/catabolism pgl PA3182 6-phosphogluconolactonase 0,64 1,71 5,53 CCM/catabolism zwf PA3183 Glucose-6-phosphate 1-dehydrogenase 1,18 1,63 5,83 Transport oprB PA3186 Glucose/carbohydrate outer membrane porin OprB precursor 2,35 2,76 5,5 Transport gltK PA3187 Probable ATP-binding component...”',
          'title': 'The development of a new parameter for tracking post-transcriptional regulation allows the detailed map of the Pseudomonas aeruginosa Crc regulon',
          'journal': 'Scientific reports 2018',
          'authors': 'Corona F, Reales-Calderón JA, Gil C, Martínez JL.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632916',
          'text': '“...PA1777 PA1802 PA2023 PA2397 PA2653 PA2853 PA2854 PA3078 PA3182 PA3257 PA3553 PA3556 PA3737 PA4020 Gene name and/or function Gutu et al. tional major peaks...”',
          'title': 'Polymyxin resistance of Pseudomonas aeruginosa phoQ mutants is dependent on additional two-component regulatory systems',
          'journal': 'Antimicrobial agents and chemotherapy 2013',
          'authors': 'Gutu AD, Sgambati N, Strasbourger P, Brannon MK, Jacobs MA, Haugen E, Kaul RK, Johansen HK, Høiby N, Moskowitz SM.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1169510',
          'text': '“...PA1317 PA1319 PA1404 PA1431 PA1432 PA2365 PA2445 PA2446 PA3181 PA3182 PA3183 PA3188 PA3190 Gene name GENE EXPRESSION IN A P. AERUGINOSA CF EPIDEMIC STRAIN VOL....”',
          'title': 'A cystic fibrosis epidemic strain of Pseudomonas aeruginosa displays enhanced virulence and antimicrobial resistance',
          'journal': 'Journal of bacteriology 2005',
          'authors': 'Salunkhe P, Smart CH, Morgan JA, Panagea S, Walshaw MJ, Hart CA, Geffers R, Tümmler B, Winstanley C.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC151497',
          'text': '“...PA2717 PA2747 PA2927 PA2939 PA3022 PA3032 PA3104 PA3181 PA3182 PA3183 PA3188 PA3189 PA3190 PA3194 PA3195 PA3274 PA3311 Description b VOL. 185, 2003...”',
          'title': 'Identification, timing, and signal specificity of Pseudomonas aeruginosa quorum-controlled genes: a transcriptome analysis',
          'journal': 'Journal of bacteriology 2003',
          'authors': 'Schuster M, Lostroh CP, Ogi T, Greenberg EP.'}],
      'url': 'http://www.brenda-enzymes.org/sequences.php?AC=Q9X2N2',
      'db': 'BRENDA',
      'gene_id': 'Q9X2N2',
      'identity': 0.27,
      'coverage': 0.87},
    'Pro0844': {'organism': 'Prochlorococcus marinus str. SS120',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2565063',
        'text': '“...2.7.1.2 Pro1065 Glucose-6-phosphate isomerase pgi 5.3.1.9 Pro0946 Glucose-6-phosphate dehydrogenase zwf 1.1.1.49 Pro1124 6-phosphogluconolactonase nagB 3.1.1.31 Pro0844 6-phosphogluconate dehydrogenase gnd 1.1.1.44 Pro0843 Phosphoribulokinase prk 2.7.1.19 Pro0861 Pentose-5-phosphate-3-epimerase rpe 5.1.3.1 Pro0839 Ribose-5-phosphate isomerase rpi 5.3.1.6 Pro1644 Transketolase tktA 2.2.1.1 Pro1770 Transaldolase tal 2.2.1.2 Pro0519 Triosephosphate isomerase tpi 5.3.1.1...”',
        'title': 'Glucose uptake and its effect on gene expression in prochlorococcus',
        'journal': 'PloS one 2008',
        'authors': 'Gómez-Baena G, López-Lozano A, Gil-Martínez J, Lucena JM, Diez J, Candau P, García-Fernández JM.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=564268',
      'db': 'MicrobesOnline',
      'gene_id': '564268',
      'identity': 0.3,
      'coverage': 0.84},
    'TC0458': {'organism': 'Chlamydia muridarum Nigg',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3019893',
        'text': '“...TC0422 TC0425 TC0437 TC0438 TC0439 TC0446 TC0448 TC0449 TC0458 TC0473 TC0478 TC0479 TC0491 TC0500 TC0503 TC0504 TC0512 TC0519 TC0521 TC0522 TC0527 TC0536 TC0539...”',
        'title': 'Identification of immunodominant antigens by probing a whole Chlamydia trachomatis open reading frame proteome microarray using sera from immunized mice',
        'journal': 'Infection and immunity 2011',
        'authors': 'Cruz-Fisher MI, Cheng C, Sun G, Pal S, Teng A, Molina DM, Kayala MA, Vigil A, Baldi P, Felgner PL, Liang X, de la Maza LM.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=370773',
      'db': 'MicrobesOnline',
      'gene_id': '370773',
      'identity': 0.3,
      'coverage': 0.71},
    'BB0222': {'organism': 'Borrelia burgdorferi B31',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2651281',
        'text': '“...from bb0220 by 40 bp, bb0223 is separated from bb0222 by 20 bp, and no intergenic regions are predicted between the other adjacent genes (22). To determine...”',
        'title': 'A manganese transporter, BB0219 (BmtA), is required for virulence by the Lyme disease spirochete, Borrelia burgdorferi',
        'journal': 'Proceedings of the National Academy of Sciences of the United States of America 2009',
        'authors': 'Ouyang Z, He M, Oman T, Yang XF, Norgard MV.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1698074',
          'text': '“...including genes that encode a putative glucose-6-phosphate-1-dehydrogenase (BB0222), an alanyl t-RNA synthetase (BB0220), and a putative zinc permease (BB0219)....”',
          'title': 'Identification of potential virulence determinants by Himar1 transposition of infectious Borrelia burgdorferi B31',
          'journal': 'Infection and immunity 2006',
          'authors': 'Botkin DJ, Abbott AN, Stewart PE, Rosa PA, Kawabata H, Watanabe H, Norris SJ.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=43864',
      'db': 'MicrobesOnline',
      'gene_id': '43864',
      'identity': 0.3,
      'coverage': 0.79},
    'PPUBIRD1_1073': {'organism': 'Pseudomonas putida BIRD-1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4720416',
        'text': '“...genes involved in pentose phosphate pathways, such as Zwf, Edd and PgI (PPUBIRD1_1071, PPUBIRD1_1060 and PPUBIRD1_1073, respectively) were found to be strongly downregulated when butanol was used as sole carbon source. Membrane proteome involved in butanol assimilation QedH protein abundance was strongly upregulated (41.5fold change) in...”',
        'title': 'Understanding butanol tolerance and assimilation in Pseudomonas putida BIRD-1: an integrated omics approach',
        'journal': 'Microbial biotechnology 2016',
        'authors': 'Cuenca Mdel S, Roca A, Molina-Santiago C, Duque E, Armengaud J, Gómez-Garcia MR, Ramos JL.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_041166853.1',
      'db': 'RefSeq',
      'gene_id': 'WP_041166853.1',
      'identity': 0.25,
      'coverage': 0.89},
    'EIO_1651, KVC_1673, KVH_07945, KvSKV_07915': {'organism': 'Ketogulonicigenium vulgare',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6293013',
        'text': '“...KVH_eda* KVU_PB0157 EIO_3350 Zwf PPP 1.1.1.49 KVC_1674 KvSKV_07920 KVH_07950 KVU_1124 EIO_1652 Pgl PPP 3.1.1.31 KVC_1673 KvSKV_07915 KVH_07945 KVU_1123 EIO_1651 Gnd PPP 1.1.1.44 KVC_0890 KvSKV_03940 KVH_03970 KVU_0365 EIO_0834 RpiA PPP 5.3.1.6 KVC_0880 KvSKV_03890 KVH_03920 KVU_0356 EIO_0823 Rpe PPP 5.1.3.1 KVC_2798 KvSKV_03940 KVH_13715 KVU_2194 EIO_2692 TktA PPP 2.2.1.1...”',
        'title': 'Reconstruction and analysis of carbon metabolic pathway of Ketogulonicigenium vulgare SPU B805 by genome and transcriptome',
        'journal': 'Scientific reports 2018',
        'authors': 'Wang CY, Li Y, Gao ZW, Liu LC, Wu YC, Zhang MY, Zhang TY, Zhang YX.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6293013',
          'text': '“...EIO_3350 Zwf PPP 1.1.1.49 KVC_1674 KvSKV_07920 KVH_07950 KVU_1124 EIO_1652 Pgl PPP 3.1.1.31 KVC_1673 KvSKV_07915 KVH_07945 KVU_1123 EIO_1651 Gnd PPP 1.1.1.44 KVC_0890 KvSKV_03940 KVH_03970 KVU_0365 EIO_0834 RpiA PPP 5.3.1.6 KVC_0880 KvSKV_03890 KVH_03920 KVU_0356 EIO_0823 Rpe PPP 5.1.3.1 KVC_2798 KvSKV_03940 KVH_13715 KVU_2194 EIO_2692 TktA PPP 2.2.1.1 KVC_2247 KvSKV_10835...”',
          'title': 'Reconstruction and analysis of carbon metabolic pathway of Ketogulonicigenium vulgare SPU B805 by genome and transcriptome',
          'journal': 'Scientific reports 2018',
          'authors': 'Wang CY, Li Y, Gao ZW, Liu LC, Wu YC, Zhang MY, Zhang TY, Zhang YX.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_013384425.1',
      'db': 'RefSeq',
      'gene_id': 'WP_013384425.1',
      'identity': 0.28,
      'coverage': 0.84},
    'ZMO1478': {'organism': 'Zymomonas mobilis subsp. mobilis ZM4',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6190654',
        'text': '“...metabolism ZMO0070, ZMO1422, ZMO1593 Posttranslational modification, protein turnover, and chaperones ZMO0472, ZMO1124, Rho Transcription Pgi, ZMO1478 Carbohydrate transport and metabolism Mfd, ZMO0127 Replication, recombination, and repair ZMO1439, ZMO1522 Cell wall/membrane/envelope biogenesis ZMO0495, ZMO0912 Uncharacterized protein ZMO1754, ZMO1113 Energy production and conversion ZMO1485 Nucleotide transport and metabolism...”',
        'title': 'Proteomic and metabolomic analysis of the cellular biomarkers related to inhibitors tolerance in Zymomonas mobilis ZM4',
        'journal': 'Biotechnology for biofuels 2018',
        'authors': 'Chang D, Yu Z, Ul Islam Z, French WT, Zhang Y, Zhang H.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5770649',
          'text': '“...linkage of lactone, resulting in production of 6-phophogluconate in the pentose phosphate pathway. Up-regulation of ZMO1478 (6-phosphogluconolactonase) transcript in acetate stress has been previously reported (Yang et al., 2014a ), and in this study, UTR_ZMO1478 (6-phosphogluconolactonase) was shown to activate GFP expression under acetate stress (10...”',
          'title': "Identification and Characterization of 5' Untranslated Regions (5'UTRs) in Zymomonas mobilis as Regulatory Biological Parts",
          'journal': 'Frontiers in microbiology 2017',
          'authors': 'Cho SH, Haning K, Shen W, Blome C, Li R, Yang S, Contreras LM.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4578398',
          'text': '“...ZMO0997 (KDPG aldolase), ZMO1237 ( d -isomer specific 2-hydroxyacid dehydrogenase), ZMO1307 (fumarase), ZMO1360 (pyruvate decarboxylase), ZMO1478 (6-phosphogluconolactonase), ZMO1496 (phosphoenolpyruvate carboxylase), ZMO1596 (Fe-containing alcohol dehydrogenase), ZMO1608 (phosphopyruvate hydratase), and ZMO1963 (citrate synthase). The primers for qRT-PCR of the genes are listed in Additional file 1 Our hierarchical...”',
          'title': 'Transcriptome analysis of Zymomonas mobilis ZM4 reveals mechanisms of tolerance and detoxification of phenolic aldehyde inhibitors from lignocellulose pretreatment',
          'journal': 'Biotechnology for biofuels 2015',
          'authors': 'Yi X, Gu H, Gao Q, Liu ZL, Bao J.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033097',
          'text': '“...mostly related to the central carbon metabolism, which included genes such as 6-phosphogluconolactonase gene pgl (ZMO1478), phosphoglycerate mutase gene pgmA (ZMO1240), gluconolactonase gene gnl (ZMO1649), pyruvate decarboxylase gene pdC (ZMO1360), and pyruvate dehydrogenase gene pdhA and pdhB (ZMO1606, ZMO1605) (Additional Files 13 , 14B ). Genes...”',
          'title': 'Elucidation of Zymomonas mobilis physiology and stress responses by quantitative proteomics and transcriptomics',
          'journal': 'Frontiers in microbiology 2014',
          'authors': 'Yang S, Pan C, Hurst GB, Dice L, Davison BH, Brown SD.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3495753',
          'text': '“...glf 0.92 ZMO0369 glk (glucokinase) 0.86 ZMO0367 zwf 0.79 ZMO1649 gnl 0.34 ZMO1757 gntK 0.48 ZMO1478 pgl 0.67 ZMO0368 edd 0.89 ZMO0997 eda 0.88 ZMO0177 gap 0.97 ZMO0178 pgk 0.8 ZMO1240 gpm 0.91 ZMO1608 eno 0.87 ZMO0152 pyk 0.87 ZMO1496 Ppc 0.64 ZMO1237 ldhA 0.54 ZMO1360...”',
          'title': 'Transcriptome profiling of Zymomonas mobilis under ethanol stress',
          'journal': 'Biotechnology for biofuels 2012',
          'authors': 'He MX, Wu B, Shui ZX, Hu QC, Wang WG, Tan FR, Tang XY, Zhu QL, Pan K, Li Q, Su XH.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6870993',
          'text': '“...zwf , ZMO0367) oxidizes glucose-6-phosphate to 6-phosphonolactone. The lactone is dehydrated to 6-phosphogluconate by lactonase (ZMO1478). 6-phosphogluconate is dehydrated by 6-phosphogluconate dehydratase ( edd , ZMO0368) to yield 2-keto-3-deoxy-6-phosphogluconate (KDPG). KDPG aldolase ( eda , ZMO0997) cleaves KDPG to form pyruvate and glyceraldehyde-3-phosphate ( Fig. 2...”',
          'title': 'The genome sequence of the ethanologenic bacterium Zymomonas mobilis ZM4',
          'journal': 'Nature biotechnology 2005',
          'authors': 'Seo JS, Chong H, Park HS, Yoon KO, Jung C, Kim JJ, Hong JH, Kim H, Kim JH, Kil JI, Park CJ, Oh HM, Lee JS, Jin SJ, Um HW, Lee HJ, Oh SJ, Kim JY, Kang HL, Lee SY, Lee KJ, Kang HS.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=958284',
      'db': 'MicrobesOnline',
      'gene_id': '958284',
      'identity': 0.3,
      'coverage': 0.71},
    'AMETH_4097': {'organism': 'Amycolatopsis methanolica 239',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640789',
        'text': '“...AMETH_4093 AMED_2808 AORI_2795 AJAP_24815 pgi AMETH_4094 AMED_2807 AORI_2794 AJAP_24820 zwf AMETH_4095 AMED_2806 AORI_2793 AJAP_24825 pgl AMETH_4097 AMED_2804 AORI_2791 AJAP_24835 Glycolysis/gluconeogenesis glpX AMETH_5852 AMED_8059 AORI_6881 AJAP_04695 AMETH_4516 AMETH_4530 fbaA AMETH_6838 AMED_9110 AORI_7850 AJAP_40945 AMETH_4655 AMETH_6121 AMED_3923 AORI_5186 AJAP_13540 AMETH_4531 Each line is the corresponding ortholog. On the...”',
        'title': 'A systematic study of the whole genome sequence of Amycolatopsis methanolica strain 239T provides an insight into its physiological and taxonomic properties which correlate with its position in the genus',
        'journal': 'Synthetic and systems biotechnology 2016',
        'authors': 'Tang B, Xie F, Zhao W, Wang J, Dai S, Zheng H, Ding X, Cen X, Liu H, Yu Y, Zhou H, Zhou Y, Zhang L, Goodfellow M, Zhao GP.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_020421295.1',
      'db': 'RefSeq',
      'gene_id': 'WP_020421295.1',
      'identity': 0.26,
      'coverage': 0.82},
    'CT186': {'organism': 'Chlamydia trachomatis D/UW-3/CX',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427450',
        'text': '“...pathway CT063 gnd CT121 araA CT185 zwf CT186 devB CT213 rpiA CT313 tal CT331 dxs CT750 tktB 6-Phosphogluconate dehydrogenase Ribulose-3-phosphate epimerase...”',
        'title': 'Chlamydia trachomatis lacks an adaptive response to changes in carbon source availability',
        'journal': 'Infection and immunity 2004',
        'authors': 'Nicholson TL, Chiu K, Stephens RS.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=189',
      'db': 'MicrobesOnline',
      'gene_id': '189',
      'identity': 0.28,
      'coverage': 0.86},
    'NMA1608': {'organism': 'Neisseria meningitidis Z2491',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1418650',
        'text': '“...NMB1961 NMA0227 NMA0228 NMA0229 NMA0230 NMA0233 NMA1609 NMA1608 NMA1607 NMA1605 NMA1604 NMA0493 NMA0492 NMA0491 NMA0490 NMB1962 NMB1963 NMB1964 NMB1965 NMB1966...”',
        'title': 'Identification of a meningococcal L-glutamate ABC transporter operon essential for growth in low-sodium environments',
        'journal': 'Infection and immunity 2006',
        'authors': 'Monaco C, Talà A, Spinosa MR, Progida C, De Nitto E, Gaballo A, Bruni CB, Bucci C, Alifano P.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1418650',
          'text': '“...Z2491) NMB2141 NMB2142 NMB2143 NMB2144 NMB2147 NMB1392 NMB1391 NMB1390 NMB1389 NMB1388 NMB1958 NMB1959 NMB1960 NMB1961 NMA0227 NMA0228 NMA0229 NMA0230 NMA0233...”',
          'title': 'Identification of a meningococcal L-glutamate ABC transporter operon essential for growth in low-sodium environments',
          'journal': 'Infection and immunity 2006',
          'authors': 'Monaco C, Talà A, Spinosa MR, Progida C, De Nitto E, Gaballo A, Bruni CB, Bucci C, Alifano P.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=142675',
      'db': 'MicrobesOnline',
      'gene_id': '142675',
      'identity': 0.26,
      'coverage': 0.76},
    'H16_B2565': {'organism': 'Cupriavidus necator H16',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6572756',
        'text': '“...or no changes, none of which were statistically significant. These included the genes for 6-phosphogluconolactonase (H16_B2565, no change), 6-phosphogluconate dehydratase (H16_B2567, no change) and 2-keto-3-deoxy-6-phosphogluconate aldolase (B16_B1213, 3.4-fold downregulated; p adj >0.05) (Additional file 3 : Table S1). A second 6-phosphogluconate dehydratase gene, H16_A1178, was strongly...”',
        'title': 'The genetic basis of 3-hydroxypropanoate metabolism in Cupriavidus necator H16',
        'journal': 'Biotechnology for biofuels 2019',
        'authors': 'Arenas-López C, Locker J, Orol D, Walter F, Busche T, Kalinowski J, Minton NP, Kovács K, Winzer K.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_011617961.1',
      'db': 'RefSeq',
      'gene_id': 'WP_011617961.1',
      'identity': 0.29,
      'coverage': 0.76},
    'RSP_2735': {'organism': 'Rhodobacter sphaeroides 2.4.1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4323418',
        'text': '“...Fig.4 from reference 28 . pgi (glucose-6-phosphate isomerase; RSP_2736), zwf (glucose-6-phosphate 1-dehydrogenase; RSP_2734), pgl (6-phosphogluconolactonase; RSP_2735), edd (6-phosphogluconate dehydratase; RSP_2646), eda (KDPG/KHG bifunctional aldolase; RSP_2645), fbaB (fructose-bisphosphate aldolase; RSP_4045), pckA (phosphoenolpyruvate carboxykinase; RSP_1680), sucAB (-ketoglutarate dehydrogenase; RSP_0965-4), sucCD (succinyl-CoA synthetase; RSP_0967-6), sdh (succinate dehydrogenase; RSP_0974-9), atpHAGDC...”',
        'title': 'CceR and AkgR regulate central carbon and energy metabolism in alphaproteobacteria',
        'journal': 'mBio 2015',
        'authors': 'Imam S, Noguera DR, Donohue TJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4344238',
          'text': '“...enzyme (RSP_1593), PdhAB (RSP_2968-RSP_4047-RSP_4050), succinate dehydrogenase (RSP_09746), as well as glycolytic enzymes Zwf (RSP_2734), Pgl (RSP_2735), Pgi (RSP_2736) and FbaB (RSP_4045), potentially making this TF a major regulator of carbon metabolism under many conditions ( Fig. 6 ). This predicted RSP_1663 regulon might make it functionally...”',
          'title': 'An integrated approach to reconstructing genome-scale transcriptional regulatory networks',
          'journal': 'PLoS computational biology 2015',
          'authors': 'Imam S, Noguera DR, Donohue TJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441632',
          'text': '“...Entner-Doudoroff RSP_2646 Fermentation RSP_3164 Glycolysis/gluconeogenesis RSP_2736, RSP_4045, RSP_4211 Other RSP_0392, RSP_2294 Pentose phosphate pathway RSP_2734, RSP_2735 Sugars RSP_2937, RSP_3138 Biosynthesis of cofactors, prosthetic groups, and carriers Glutathione and analogs RSP_3272 Heme, porphyrin, and cobalamin RSP_1197, RSP_1692, RSP_2831 Menaquinone and ubiquinone RSP_1175, RSP_1338, RSP_1492, RSP_1869 Other RSP_0750,...”',
          'title': 'Convergence of the transcriptional responses to heat shock and singlet oxygen stresses',
          'journal': 'PLoS genetics 2012',
          'authors': 'Dufour YS, Imam S, Koo BM, Green HA, Donohue TJ.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=1139795',
      'db': 'MicrobesOnline',
      'gene_id': '1139795',
      'identity': 0.27,
      'coverage': 0.78},
    'SO2488': {'organism': 'Shewanella oneidensis MR-1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2980473',
        'text': '“...3 3 2 3 4 SO2486 CN32_1869 eda SO2487 CN32_1868 edd 2 2 2 3 SO2488 CN32_1867 pgl SO2489 CN32_1866 zwf SO2491 CN32_1864 pyk A 3 8 4 6 5 3 5 SO2644 CN32_2243 pps A 11 7 9 8 8 4 5 3 SO3440 CN32_2757...”',
        'title': 'An empirical strategy for characterizing bacterial proteomes across species in the absence of genomic sequences',
        'journal': 'PloS one 2010',
        'authors': 'Turse JE, Marshall MJ, Fredrickson JK, Lipton MS, Callister SJ.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1482949',
          'text': '“...so2487 (edd, 6-phosphogluconate dehydratase; 2.6-fold), so2488 (pgl, 6-phosphogluconolactonase; 2.9-fold), and so2489 (zwf, glucose-6-phosphate 1-dehydrogenase;...”',
          'title': 'Global transcriptome analysis of the cold shock response of Shewanella oneidensis MR-1 and mutational analysis of its classical cold shock proteins',
          'journal': 'Journal of bacteriology 2006',
          'authors': 'Gao H, Yang ZK, Wu L, Thompson DK, Zhou J.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=201629',
      'db': 'MicrobesOnline',
      'gene_id': '201629',
      'identity': 0.26,
      'coverage': 0.85},
    'BCAL3043': {'organism': 'Burkholderia cenocepacia J2315',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5688073',
        'text': '“...e.g. gluconate permease (BCAL3365), galactonate transporter (BCAL0184, BCAM2500), glycerol kinase (BCAL0925), glycerol-phosphate transporter (BCAL3041), 6-phosphogluconolactonase (BCAL3043) and salicylate hydroxylase(BCAM1274). Predicted targets of ncS16 include a higher percentage of outer membrane and cell envelope components, as well as genes for transport of inorganic compounds. The predicted targets...”',
        'title': 'Identification of small RNAs abundant in Burkholderia cenocepacia biofilms reveal putative regulators with a potential role in carbon and iron metabolism',
        'journal': 'Scientific reports 2017',
        'authors': 'Sass A, Kiekens S, Coenye T.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=5925399',
      'db': 'MicrobesOnline',
      'gene_id': '5925399',
      'identity': 0.22,
      'coverage': 0.94},
    'Rmet_5800': {'organism': 'Ralstonia metallidurans CH34',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2864759',
        'text': '“...to 6-phosphogluconate (6GP) in a two-step reaction involving the pgl and zwf genes (encoded by Rmet_5800 and Rmet_5801, respectively). The 6GP is metabolized through the alternative glycolytic Entner-Doudoroff (ED) pathway involving a 6-phosphogluconate dehydratase (EDD; Rmet_5802) and a 2-keto-3-deoxy-6-phosphogluconate aldolase (EDA; Rmet_4768), thus generating two pyruvate...”',
        'title': 'The complete genome sequence of Cupriavidus metallidurans strain CH34, a master survivalist in harsh and anthropogenic environments',
        'journal': 'PloS one 2010',
        'authors': 'Janssen PJ, Van Houdt R, Moors H, Monsieurs P, Morin N, Michaux A, Benotmane MA, Leys N, Vallaeys T, Lapidus A, Monchy S, Médigue C, Taghavi S, McCorkle S, Dunn J, van der Lelie D, Mergeay M.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=1778177',
      'db': 'MicrobesOnline',
      'gene_id': '1778177',
      'identity': 0.27,
      'coverage': 0.76},
    'NE1691': {'organism': 'Nitrosomonas europaea ATCC 19718',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2546632',
        'text': '“...103 Carbohydrate transport and metabolism, NE1691 Phosphogluconolactonase/glucosamine-6-phosphate isomerase/deaminase 3.8 6.2 104 Fatty acid biosynthesis,...”',
        'title': 'Candidate stress genes of Nitrosomonas europaea for monitoring inhibition of nitrification by heavy metals',
        'journal': 'Applied and environmental microbiology 2008',
        'authors': 'Park S, Ely RL.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=368371',
      'db': 'MicrobesOnline',
      'gene_id': '368371',
      'identity': 0.26,
      'coverage': 0.65},
    'C7W88_RS12355': {'organism': 'Novosphingobium sp. THN1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6691742',
        'text': '“...0.87915 1.302 ribose-phosphate pyrophosphokinase prsA C7W88_RS01250 2.4087 2.743 phosphoglucomutase pgm C7W88_RS11010 0.78001 1.1774 6-phosphogluconolactonase devB C7W88_RS12355 2.1998 2.9099 amino sugar and nucleotide sugar metabolism UDP-N-acetyl-D-mannosaminuronic acid dehydrogenase wecC C7W88_RS02750 0.52148 0.51175 phosphoglucosamine mutase glmM C7W88_RS03760 0.61432 0.51499 fructokinase scrK C7W88_RS12310 2.1873 1.8955 UDP-N-acetyl-D-galactosamine dehydrogenase wbpO C7W88_RS02740...”',
        'title': 'Microcystin-LR Degradation and Gene Regulation of Microcystin-Degrading Novosphingobium sp. THN1 at Different Carbon Concentrations',
        'journal': 'Frontiers in microbiology 2019',
        'authors': 'Wang J, Wang C, Li Q, Shen M, Bai P, Li J, Lin Y, Gan N, Li T, Zhao J.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_118073754.1',
      'db': 'RefSeq',
      'gene_id': 'WP_118073754.1',
      'identity': 0.29,
      'coverage': 0.7},
    'CPA40_RS06275': {'organism': 'Bifidobacterium callitrichos',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096940',
        'text': '“...CPA40_RS10465 BCAL_RS06425 UDP- N -acetylglucosamine diphosphorylase 2.7.7.23 COO72_RS04345 CPA40_RS08380 BCAL_RS00645 N -Acetylglucosamine-6-phosphate deacetylase 3.5.1.25 COO72_RS10300 CPA40_RS06275 BCAL_RS02715 Glucosamine-1-phosphate N -acetyltransferase 2.3.1.157 COO72_RS04345 CPA40_RS08380 BCAL_RS00645 Phosphoglucosamine mutase 5.4.2.10 COO72_RS05530 CPA40_RS04580 BCAL_RS02085 Glucosamine-6-phosphate deaminase 3.5.99.6 COO72_RS08105 CPA40_RS05730 BCAL_RS02710 Glutaminefructose-6-phosphate transaminase (isomerizing) 2.6.1.16 COO72_RS05020 CPA40_RS04675 BCAL_RS02180 Non-reducing end -...”',
        'title': 'The comparative genomics of Bifidobacterium callitrichos reflects dietary carbohydrate utilization within the common marmoset gut',
        'journal': 'Microbial genomics 2018',
        'authors': 'Albert K, Rani A, Sela DA.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_107044159.1',
      'db': 'RefSeq',
      'gene_id': 'WP_107044159.1',
      'identity': 0.31,
      'coverage': 0.66},
    'WP_007113049': {'organism': 'Halomonas boliviensis LC1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4522284',
        'text': '“...aldolase 4.1.2.14 WP_007113045 0800 1 - - - WP_007113048 2 + + + 6-phosphogluconolactonase 3.1.1.31 WP_007113049 0363 1 + + + Glucose-6-phosphate 1-dehydrogenase 1.1.1.49 WP_007113050 0364 1 + + + Transketolase 2.2.1.1 WP_007112132 0021 1 + + + Transaldolase 2.2.1.2 WP_007111145 0176 1 - + +...”',
        'title': 'Production of poly(3-hydroxybutyrate) by Halomonas boliviensis in an air-lift reactor',
        'journal': 'Journal of biological research (Thessalonike, Greece) 2015',
        'authors': 'Rivera-Terceros P, Tito-Claros E, Torrico S, Carballo S, Van-Thuoc D, Quillaguamán J.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_007113049.1',
      'db': 'RefSeq',
      'gene_id': 'WP_007113049.1',
      'identity': 0.26,
      'coverage': 0.71},
    'lpg0417': {'organism': 'Legionella pneumophila subsp. pneumophila str. Philadelphia 1',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5928044',
        'text': '“...GDP mannose pyrophosphorylase (mannose-1-phosphate guanylyltransferase/mannose-6-phosphate isomerase) rfbA 1.57 2.14 Lpg0939 2-dehydro-3-deoxy-phosphogluconate aldolase eda 1.82 1.71 Lpg0417 6-phosphogluconolactonase pgl 1.91 2.46 Lpg0805 phosphoenolpyruvate synthase 2.32 2.85 Lpg2352 malate dehydrogenase mdh 1.39 2.28 Lpg2792 triosephosphate isomerase (TIM) tpiA 2.72 2.66 Lpg0138 glyceraldehyde 3-phosphate dehydrogenase gap 1.44 2.40 Cell...”',
        'title': 'The LetA/S two-component system regulates transcriptomic changes that are essential for the culturability of Legionella pneumophila in water',
        'journal': 'Scientific reports 2018',
        'authors': 'Mendis N, McBride P, Saoud J, Mani T, Faucher SP.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=756881',
      'db': 'MicrobesOnline',
      'gene_id': '756881',
      'identity': 0.24,
      'coverage': 0.88},
    'lpp0484': {'organism': 'Legionella pneumophila str. Paris',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5338858',
        'text': '“...Glycerol-3-phosphate dehydrogenase, GlpK (Lpp1369), glycerol kinase, Tpi (Lpp2838), Triosephosphate isomerase, Zwf (Lpp0483), Glucose-6-phosphate 1-dehydrogenase, Pgl (Lpp0484), 6-Phosphogluconolactonase, Edd (Lpp0485), 6-Phosphogluconate dehydratase, Gap (Lpp0153), Glyceraldehyde 3-phosphate dehydrogenase, Pgk (Lpp0152), 3-Phosphoglycerate kinase, Eno (Lpp2020), Enolase, Pyk (Lpp0151), Pyruvate kinase, PpsA (Lpp0567), Phosphoenolpyruvate synthase, Pdh (Lpp1461), Pyruvate dehydrogenase complex,...”',
        'title': 'The Legionella pneumophila genome evolved to accommodate multiple regulatory mechanisms controlled by the CsrA-system',
        'journal': 'PLoS genetics 2017',
        'authors': 'Sahr T, Rusniok C, Impens F, Oliva G, Sismeiro O, Coppée JY, Buchrieser C.'},
        {'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2903384',
          'text': '“...transcripts were determined via RT-PCR. lpp0483, zwf, lpp0484, pgl; lpp0485, edd; lpp0486, glk; lpp0487, eda; lpp0488 (putative sugar transport protein);...”',
          'title': 'Isotopologue profiling of Legionella pneumophila: role of serine and glucose as carbon substrates',
          'journal': 'The Journal of biological chemistry 2010',
          'authors': 'Eylert E, Herrmann V, Jules M, Gillmaier N, Lautner M, Buchrieser C, Eisenreich W, Heuner K.'}],
      'url': 'http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=753824',
      'db': 'MicrobesOnline',
      'gene_id': '753824',
      'identity': 0.28,
      'coverage': 0.62},
    'BCAL_RS02710': {'organism': 'Bifidobacterium callitrichos DSM 23973',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096940',
        'text': '“...2.3.1.157 COO72_RS04345 CPA40_RS08380 BCAL_RS00645 Phosphoglucosamine mutase 5.4.2.10 COO72_RS05530 CPA40_RS04580 BCAL_RS02085 Glucosamine-6-phosphate deaminase 3.5.99.6 COO72_RS08105 CPA40_RS05730 BCAL_RS02710 Glutaminefructose-6-phosphate transaminase (isomerizing) 2.6.1.16 COO72_RS05020 CPA40_RS04675 BCAL_RS02180 Non-reducing end - l -arabinofuranosidase 3.2.1.55 COO72_RS05470 CPA40_RS04135 BCAL_RS02145 Glucose-6-phosphate isomerase 5.3.1.9 COO72_RS12210 CPA40_RS08670 BCAL_RS05265 UDP-glucose 6-dehydrogenase 1.1.1.22 COO72_RS07800 CPA40_RS09840 BCAL_RS07900 Phosphoglucomutase (-...”',
        'title': 'The comparative genomics of Bifidobacterium callitrichos reflects dietary carbohydrate utilization within the common marmoset gut',
        'journal': 'Microbial genomics 2018',
        'authors': 'Albert K, Rani A, Sela DA.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/WP_043164840.1',
      'db': 'RefSeq',
      'gene_id': 'WP_043164840.1',
      'identity': 0.29,
      'coverage': 0.66},
    'TGME49_078830': {'organism': 'Toxoplasma gondii ME49',
      'papers': [{'url': 'http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3757501',
        'text': '“...aldolase PFI1340w fumarate hydratase, putative TGME49_067330 fumarase, putative (Mitochondrial localization) PF14_0511 glucose-6-phosphate dehydrogenase-6- phosphogluconolactonase (G6PDH) TGME49_078830 glucose-6-phosphate dehydrogenase, putative PF08_0132 glutamate dehydrogenase, putative (GDHc) TGME49_049390 NAD-specific glutamate dehydrogenase, putative PF13_0257 glutamate--tRNA ligase, putative TGME49_063870 glutamyl-tRNA synthetase, putative PF14_0598 glyceraldehyde-3-phosphate dehydrogenase (GAPDH) TGME49_089690; TGME49_069190 glyceraldehyde-3-phosphate dehydrogenase PF11_0183...”',
        'title': 'Extensive lysine acetylation occurs in evolutionarily conserved metabolic pathways and parasite-specific functions during Plasmodium falciparum intraerythrocytic development',
        'journal': 'Molecular microbiology 2013',
        'authors': 'Miao J, Lawrence M, Jeffers V, Zhao F, Parker D, Ge Y, Sullivan WJ, Cui L.'}],
      'url': 'http://www.ncbi.nlm.nih.gov/protein/XP_018635111.1',
      'db': 'RefSeq',
      'gene_id': 'XP_018635111.1',
      'identity': 0.36,
      'coverage': 0.09}}

  let w = new WidgetPaperBlast(e, $('#test_container1'));
  w.init_container();

  let icons = {
    'EV-EXP' : '<i class="fas fa-flask"></i>',
    'EV-EXP-IDA' : '<i class="fas fa-vial"></i>',
    'EV-COMP-HINF-SEQ-ORTHOLOGY' : '<i class="fas fa-laptop"></i>',
    'EV-COMP-HAINF-FN-FROM-SEQ' : '<i class="fas fa-user-check"></i><i class="fas fa-laptop"></i>',
    'a' : '<i class="fas fa-pastafarianism"></i>', //mutant
    'ban' : '<i class="fas fa-ban"></i>',
    'EV-EXP-IMP-REACTION-BLOCKED' : '<i class="fas fa-pastafarianism"></i><i class="fas fa-ban"></i>'
  };
  function formatSelection(state) {
    console.log('formatSelection', state);
    if (icons[state.id]) {
      let res = $('<span>').append(icons[state.id]);
      return res;
    }
    return state.text;
  }
  function formatState (state) {
    //console.log(state);
    if (!state.id) {
      return state.text;
    }

    let baseUrl = "/user/pages/images/flags";
    let $state = $(
      '<span><img src="' + baseUrl + '/' + state.element.value.toLowerCase() + '.png" class="img-flag" /> ' + state.text + '</span>'
    );
    let res = $('<span>')
    if (icons[state.id]) {
      res.append(icons[state.id]).append(' ');
    }
    res.append(state.text);
    return res;
  };

  $('.js-example-basic-multiple').select2({
    maximumSelectionLength: 5,
    placeholder: 'This is my placeholder',
    allowClear: true,
    multiple: true,
    //width: '600px',
    templateSelection: formatSelection,
    templateResult: formatState,
    data : [
      {
        "id": "EV-EXP-IMP-REACTION-BLOCKED",
        "text": "Reaction blocked in mutant"
      },
      {
        "id": "EV-EXP-IDA-PURIFIED-PROTEIN-HH",
        "text": "Assay of protein purified to homogeneity from a heterologous host",
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-PURIFIED-PROTEIN-NH",
        "text": "Assay of protein purified to homogeneity from its native host",
        "disabled": true
      },
      {
        "id": "EV-EXP-IDA-PURIFIED-PROTEIN",
        "text": "Assay of protein purified to homogeneity",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IGI-FUNC-COMPLEMENTATION",
        "text": "Inferred by functional complementation",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-UNPURIFIED-PROTEIN-HH",
        "text": "Assay of unpurified protein expressed in a heterologous host",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-UNPURIFIED-PROTEIN",
        "text": "Assay of unpurified protein",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-UNPURIFIED-PROTEIN-NH",
        "text": "Assay of unpurified protein expressed in its native host",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-PART-PURIFIED-PROTEIN-NH",
        "text": "Assay of protein partially-purified from its native host",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IMP",
        "text": "Inferred from mutant phenotype",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP",
        "text": "Inferred from experiment",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA",
        "text": "Inferred from direct assay",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-PART-PURIFIED-PROTEIN",
        "text": "Assay of partially-purified protein",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-COMP-HINF-FN-FROM-SEQ",
        "text": "Human inference of function from sequence",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IMP-REACTION-ENHANCED",
        "text": "Reaction enhanced in mutant",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IDA-PART-PURIFIED-PROTEIN-HH",
        "text": "Assay of protein partially-purified from a heterologous host",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IGI",
        "text": "Inferred from genetic interaction",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-IEP",
        "text": "Inferred from expression pattern",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-COMP-HINF-SEQ-ORTHOLOGY",
        "text": "Human inference of function by sequence orthology",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-EXP-TAS",
        "text": "Traceable author statement to experimental support",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-COMP-HAINF-FN-FROM-SEQ",
        "text": "Inferred by a human author based on computational evidence",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-COMP-AINF-FN-FROM-SEQ",
        "text": "Automated inference of function from sequence",
        "disabled": false,
        "selected": false
      },
      {
        "id": "EV-IC-HINF-FN-FROM-SEQ",
        "text": "Curator inference of function from sequence",
        "disabled": false,
        "selected": false
      }
    ]
  });
    console.log('!')
    
    e = {"Phosphohistidine swiveling domain":{"hits":[{"score":21,"source":["KEGG","K00873"]}],"id":5551086,"sources":{},"subsystems":{}},"Pyruvate kinase (EC 2.7.1.40)":{"hits":[{"score":146,"source":["KEGG","K00873"]},{"score":0,"source":["template","GramNegModelTemplate"]},{"score":0,"source":["template","GramNegModelTemplateV2"]}],"id":2280,"sources":{"BETA_RAST":[1306,1558,["RLO149_RS01040@GCF_000154785.2","COCH_RS05350@GCF_000023285.1","BN940_RS01045@GCF_000612685.1","STHE_RS09785@GCF_000024985.1","UWK_RS01550@GCF_000341395.1","SVI_RS10955@GCF_000091325.1","SMUL_RS03360@GCF_000568815.1","ARNIT_RS13900@GCF_000092245.1","PCC7424_RS11540@GCF_000021825.1","MPOP_RS23010@GCF_000019945.1"]],"KBASE_RAST":[1408,1715,["RLO149_RS01040@GCF_000154785.2","COCH_RS05350@GCF_000023285.1","BN940_RS01045@GCF_000612685.1","STHE_RS09785@GCF_000024985.1","TKV_RS08220@GCF_000763575.1","UWK_RS01550@GCF_000341395.1","SVI_RS10955@GCF_000091325.1","SMUL_RS03360@GCF_000568815.1","ARNIT_RS13900@GCF_000092245.1","COPRO5265_RS04105@GCF_000020945.1"]]},"subsystems":{"Entner-Doudoroff Pathway":{},"Glycolysis and Gluconeogenesis":{},"Glycolysis and Gluconeogenesis, including Archaeal enzymes":{},"Pyruvate metabolism I: anaplerotic reactions, PEP":{}}},"Pyruvate kinase (EC 2.7.1.40) / Phosphohistidine swiveling domain":{"hits":[{"score":21,"source":["KEGG","K00873"]}],"id":5551047,"sources":{"BETA_RAST":[247,254,["DESCA_RS08745@GCF_000214435.1","TRESU_RS02515@GCF_000195275.1","PCC7424_RS16950@GCF_000021825.1","LDB_RS03675@GCF_000056065.1","WANG_RS03905@GCF_000214785.1","SYNPCC7002_RS08305@GCF_000019485.1","A361_RS22205@GCF_000294775.2","NTHER_RS02900@GCF_000020005.1","AUO94_RS02205@GCF_001465835.2","SGLY_RS08835@GCF_000190635.1"]]},"subsystems":{}},"Pyruvate kinase family protein":{"hits":[{"score":4,"source":["KEGG","K00873"]}],"id":400511,"sources":{"BETA_RAST":[92,92,["RPE_RS15350@GCF_000014825.1","GTNG_RS15315@GCF_000015745.1","THEOS_RS11820@GCF_000309885.1","CDOO_RS12485@GCF_000767055.1","B842_RS12400@GCF_000819445.1","CKROP_RS00580@GCF_000023145.1","JI59_RS22370@GCF_000767465.1","CYAGR_RS04985@GCF_000316515.1","MFUMSOLV_RS01890@GCF_000953475.1","CE_RS13615@GCF_000011305.1"]],"KBASE_RAST":[81,81,["RPE_RS15350@GCF_000014825.1","AO081_RS23665@GCF_001308105.1","AZF01_RS00330@GCF_001578105.1","H924_RS12380@GCF_000344785.1","TERTU_RS08180@GCF_000023025.1","TOLA_RS06570@GCF_000023065.1","THEOS_RS11820@GCF_000309885.1","CCE_RS03345@GCF_000017845.1","B842_RS12400@GCF_000819445.1","A605_RS13465@GCF_000341345.1"]]},"subsystems":{}},"galacturonokinase (EC 2.7.1.44)":{"hits":[{"score":0,"source":["template","GramNegModelTemplate"]},{"score":0,"source":["template","GramNegModelTemplateV2"]}],"id":5544954,"sources":{},"subsystems":{}}}
  
  let annotation_tooltip = new EscherTooltipAnnotation(null, null, null);
  e = mock_annotation_tooltip_data_rxn41846;
  let res = annotation_tooltip.escher_tooltip_annotation(
    'rxn00002', 'seed.reaction',
    e.annotation,
    e.curation,
    e.function_rxns,
    true,
    ['rxn00002', 'seed.reaction']
  );
  $('#test_tooltip1').append(res);
  $('#omgtest').append(annotation_tooltip.build_annotation_row(
    "opt_score2", "rxn00002", "example 1.1.1.1", "test_annotation_id", {"ss1" : {}, "ss2" : {}}, {'BEST_RAST' : [1306, 1558]}, {}, "fliu", false));
  $('#omgtest').append(annotation_tooltip.build_annotation_row(
    "opt_score1", "rxn00002", "example 255.255.255.255", "test_annotation_id", {"ss1" : {}, "ss2" : {}}, {'BEST_RAST' : [1306, 1558]}, {}, "system", true));
  $('#omgtest').append(annotation_tooltip.build_annotation_row(
    "opt_score1", "rxn00002", "example 0.0.0.0", "test_annotation_id", {"ss1" : {}, "ss2" : {}}, {'BEST_RAST' : [1306, 1558]}, {}, undefined, true));

  
  //render_tooltip({'seed.reaction' : ['rxn00148']}, e, $('#test_tooltip1'), {})
  
  e = {'orthologs': {
           'cluster308' : {
               'GCF_000146045.2': {
                   'YAL038W': 'pyruvate kinase CDC19',
                   'YOR347C': 'pyruvate kinase PYK2'},
               'GCF_000226395.1': {
                   'Pc18g06000': 'hypothetical protein'},
               'GCF_000182925.2': {
                   'NCU06075': 'pyruvate kinase'}}},
       'genomes': {
           'GCF_000146045.2': {
               'name': 'Saccharomyces cerevisiae S288c',
               'source': {
                   'iMM904': [
                       {
                           'gpr': '(YAL038W or YOR347C)',
                           'genes': {
                               'YAL038W': {
                                   'ortholog_id': 'cluster308',
                                   'function': 'pyruvate kinase CDC19'},
                               'YOR347C': {'ortholog_id': 'cluster308',
        'function': 'pyruvate kinase PYK2'}}}]}},
  'GCF_000182925.2': {'name': 'Neurospora crassa OR74A',
   'source': {'iJDZ836': [{'gpr': 'NCU06075',
      'genes': {'NCU06075': {'ortholog_id': 'cluster308',
        'function': 'pyruvate kinase'}}}]}},
  'GCF_000226395.1': {'name': 'Penicillium rubens Wisconsin 54-1255',
   'source': {'iAL1006': [{'gpr': '( Pc18g06000 )',
      'genes': {'Pc18g06000': {'ortholog_id': 'cluster308',
        'function': 'hypothetical protein'}}}]}}}}
  
  render_tooltip_ortholog({'seed.reaction' : ['rxn00148']}, e, $('#test_tooltip2'), {})
    
  e = {'gene_reaction': {'GCF_000146045.2': {'YAL038W': ['r_0962'],
   'YOR347C': ['r_0962']},
  'GCF_000226395.1': {'Pc18g06000': ['R_r0016']},
  'GCF_000182925.2': {'NCU06075': ['PEPDEPHOS__45__RXN__45__R2L']},
  'GCF_000277815.2': {'EHEL_090610': []},
  'GCF_000143185.1': {'SCHCODRAFT_73013': []}},
 'orthologs': {'cluster614': {'GCF_000146045.2': {'YAL038W': 'pyruvate kinase CDC19',
    'YOR347C': 'pyruvate kinase PYK2'},
   'GCF_000226395.1': {'Pc18g06000': 'hypothetical protein'},
   'GCF_000182925.2': {'NCU06075': 'pyruvate kinase'},
   'GCF_000277815.2': {'EHEL_090610': '?'},
   'GCF_000143185.1': {'SCHCODRAFT_73013': '?'}}},
 'genomes': {'GCF_000146045.2': 
             {'name': 'Saccharomyces cerevisiae S288c',
              'source': {'iMM904': [{'rxn_id': 'R_PYK',
      'gpr': '(YAL038W or YOR347C)',
      'genes': {'YOR347C': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase PYK2'},
       'YAL038W': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase CDC19'}}}],
    'iTO977': [{'rxn_id': 'R_CDC19',
      'gpr': '(YAL038W or YOR347C)',
      'genes': {'YOR347C': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase PYK2'},
       'YAL038W': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase CDC19'}}}],
    'yeast_6.06': [{'rxn_id': 'r_0962',
      'gpr': '(YOR347C OR YAL038W)',
      'genes': {'YOR347C': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase PYK2'},
       'YAL038W': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase CDC19'}}}],
    'yeast_7.6': [{'rxn_id': 'r_0962',
      'gpr': '(YAL038W OR YOR347C)',
      'genes': {'YOR347C': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase PYK2'},
       'YAL038W': {'ortholog_id': 'cluster614',
        'function': 'pyruvate kinase CDC19'}}}]}},
  'GCF_000226395.1': {'name': 'Penicillium rubens Wisconsin 54-1255',
   'source': {'iAL1006': [{'rxn_id': 'R_r0016',
      'gpr': '( Pc18g06000 )',
      'genes': {'Pc18g06000': {'ortholog_id': 'cluster614',
        'function': 'hypothetical protein'}}}]}},
  'GCF_000182925.2': {'name': 'Neurospora crassa OR74A',
   'source': {'iJDZ836': []}}}}
  
  render_tooltip_ortholog({'seed.reaction' : ['rxn00148']}, e, $('#test_tooltip3'), {})
  
  e = {'id': 'cpd00027',
 'abbreviation': 'glc-D',
 'name': 'D-Glucose',
 'formula': 'C6H12O6',
 'mass': '180.0',
 'source': 'Primary Database',
 'inchikey': 'WQZGKKKJIJFFOK-GASJEMHNSA-N',
 'charge': 0,
 'is_core': 1,
 'is_obsolete': 0,
 'linked_compound': '',
 'is_cofactor': 0,
 'deltag': -102.13,
 'deltagerr': 0.16,
 'pka': '8:15.28;12:13.87;10:14.69;6:13.03;11:11.61',
 'pkb': '12:-3.73;10:-3.51;11:-4.26;3:-4.09;6:-3.49;8:-2.9',
 'abstract_compound': '',
 'comprised_of': '',
 'aliases': 'Name: D-(+)-Glucose; D-Glucopyranose; D-Glucose; Dextrose; Glucopyranose; Glucose; Grape sugar|AlgaGEM: S_D_45_Glucose_c; S_Glucose_c; S_Glucose_ext_b; S_Glucose_p|AraGEM: S_D_45_Glucose_c|BiGG: glc__D|KEGG: C00031; C00293|MetaCyc: Glucopyranose',
 'smiles': 'OC[C@H]1OC(O)[C@H](O)[C@@H](O)[C@@H]1O'}
  
  e = {
      'models' : ['model1', 'model2', 'model3'],
      'records' : [
          {
              'model_data' : {
                  'c' : {
                      'model1' : ['M_cpd1_c', 'M_cpd1_dup_c'],
                      'model2' : ['M_zpd1_cyt'],
                      'model3' : ['M_ccc1_cytosol'],
                  },
                  'm' : {
                      'model1' : ['M_cpd1_m'],
                      'model2' : ['M_zpd1_mit'],
                      'model3' : ['M_ccc1_mitss'],
                  }
              },
              'database' : {
                  'seed.compound': ['cpd00001', 'cpd00002']
              }
          }
      ]
  }
  render_report(e, $('#test_tooltip4'))
    
  e = {'models': ['iJDZ836', 'iMM904', 'iAL1006'],
 'records': [
     {'model_data': {'c': {'iJDZ836': ['M_2__45__KETOGLUTARATE'],
     'iMM904': ['M_akg_c'],
     'iAL1006': ['M_AKG']}},
   'database': {}},
  {'model_data': {'c': {'iJDZ836': ['M_GLT'],
     'iMM904': ['M_glu__L_c'],
     'iAL1006': ['M_GLU']}},
   'database': {}},
  {'model_data': {'c': {'iMM904': ['M_3php_c'],
     'iAL1006': ['M_PHP'],
     'iJDZ836': ['M_3__45__P__45__HYDROXYPYRUVATE']}},
   'database': {}},
  {'model_data': {'c': {'iJDZ836': ['M_WATER'],
     'iMM904': ['M_h2o_c'],
     'iAL1006': ['M_H2O']}},
   'database': {'seed.compound' : ['cpd00001']}},
  {'model_data': {'c': {'iMM904': ['M_nadh_c'],
     'iAL1006': ['M_NADH'],
     'iJDZ836': ['M_NADH']}},
   'database': {}},
  {'model_data': {'c': {'iMM904': ['M_h_c'], 'iJDZ836': ['M_PROTON']}},
   'database': {}},
  {'model_data': {'c': {'iJDZ836': ['M_GLY'],
     'iAL1006': ['M_GLY'],
     'iMM904': ['M_gly_c']}},
   'database': {}},
  {'model_data': {'c': {'iAL1006': ['M_CYS'],
     'iJDZ836': ['M_CYS'],
     'iMM904': ['M_cys__L_c']}},
   'database': {}},
  {'model_data': {'c': {'iMM904': ['M_h2s_c'],
     'iAL1006': ['M_H2S'],
     'iJDZ836': ['M_HS']}},
   'database': {}},
  {'model_data': {'c': {'iJDZ836': ['M_PROTON'], 'iMM904': ['M_h_c']}},
   'database': {}}]}
    
    
e = {
    "models": [
        "iMM904"
    ],
    "records": [
        {
            "c": {
                "iMM904": [
                    "M_atp_c"
                ]
            },
            "database": {
                "seed.compound": [
                    "cpd00002"
                ]
            },
            "model_data": {}
        },
        {
            "c": {
                "iMM904": [
                    "M_fru_c"
                ]
            },
            "database": {
                "seed.compound": [
                    "cpd00082"
                ]
            },
            "model_data": {}
        }
    ]
}
    
  render_report(e, $('#test_tooltip5'))


})
</script>
</html>